{
 "cells": [
  {
   "cell_type": "markdown",
   "id": "ea8f1e8d",
   "metadata": {},
   "source": [
    "## Data Set from Kaggle\n",
    "\n",
    "Recognized the wealth of crucial information within biomedical texts pivotal for advancements in the medical domain. \n",
    "\n",
    "Advocated for the automation of event extraction from biomedical texts to accelerate progress and innovation within the medical field, emphasizing the substantial benefits of this approach.\n",
    "\n",
    "https://www.kaggle.com/datasets/nishanthsalian/genia-biomedical-event-dataset?select=GE11-LICENSE"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "id": "7e186f1f",
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as pd"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "id": "44324855",
   "metadata": {},
   "outputs": [],
   "source": [
    "train = pd.read_csv('train_data.csv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "id": "50fb9f47",
   "metadata": {},
   "outputs": [],
   "source": [
    "test = pd.read_csv('test_data.csv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "d3af2881",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "(8666, 4)"
      ]
     },
     "execution_count": 4,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "train.shape"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "id": "c04d82ec",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "(3360, 4)"
      ]
     },
     "execution_count": 5,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "test.shape"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "id": "c9d1cc2d",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Sentence</th>\n",
       "      <th>TriggerWord</th>\n",
       "      <th>TriggerWordLoc</th>\n",
       "      <th>EventType</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>Down-regulation of interferon regulatory fact...</td>\n",
       "      <td>Down-regulation;expression;</td>\n",
       "      <td>1;8;</td>\n",
       "      <td>Negative_regulation;Gene_expression;</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>Although the bcr - abl translocation has been...</td>\n",
       "      <td>deregulation;</td>\n",
       "      <td>30;</td>\n",
       "      <td>Regulation;</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>Promoter methylation of CpG target sites or d...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>Therefore , we investigated whether IRF-4 pro...</td>\n",
       "      <td>regulation;expression;</td>\n",
       "      <td>16;19;</td>\n",
       "      <td>Regulation;Gene_expression;</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>Whereas promoter mutations or structural rear...</td>\n",
       "      <td>altered;expression;influence;transcription;</td>\n",
       "      <td>14;16;30;32;</td>\n",
       "      <td>Regulation;Gene_expression;Regulation;Transcri...</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                                            Sentence  \\\n",
       "0   Down-regulation of interferon regulatory fact...   \n",
       "1   Although the bcr - abl translocation has been...   \n",
       "2   Promoter methylation of CpG target sites or d...   \n",
       "3   Therefore , we investigated whether IRF-4 pro...   \n",
       "4   Whereas promoter mutations or structural rear...   \n",
       "\n",
       "                                   TriggerWord TriggerWordLoc  \\\n",
       "0                  Down-regulation;expression;           1;8;   \n",
       "1                                deregulation;            30;   \n",
       "2                                          NaN            NaN   \n",
       "3                       regulation;expression;         16;19;   \n",
       "4  altered;expression;influence;transcription;   14;16;30;32;   \n",
       "\n",
       "                                           EventType  \n",
       "0               Negative_regulation;Gene_expression;  \n",
       "1                                        Regulation;  \n",
       "2                                                NaN  \n",
       "3                        Regulation;Gene_expression;  \n",
       "4  Regulation;Gene_expression;Regulation;Transcri...  "
      ]
     },
     "execution_count": 6,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "train.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "id": "748c022d",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Sentence</th>\n",
       "      <th>TriggerWord</th>\n",
       "      <th>TriggerWordLoc</th>\n",
       "      <th>EventType</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>Resistance to IL-10 inhibition of interferon ...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>IL-10 has been shown to block the antigen-spe...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>We found that peripheral blood CD4 + T cells ...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>The phosphorylation of signal transducer and ...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>Sera from RA patients induced signal transduc...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                                            Sentence  TriggerWord  \\\n",
       "0   Resistance to IL-10 inhibition of interferon ...          NaN   \n",
       "1   IL-10 has been shown to block the antigen-spe...          NaN   \n",
       "2   We found that peripheral blood CD4 + T cells ...          NaN   \n",
       "3   The phosphorylation of signal transducer and ...          NaN   \n",
       "4   Sera from RA patients induced signal transduc...          NaN   \n",
       "\n",
       "   TriggerWordLoc  EventType  \n",
       "0             NaN        NaN  \n",
       "1             NaN        NaN  \n",
       "2             NaN        NaN  \n",
       "3             NaN        NaN  \n",
       "4             NaN        NaN  "
      ]
     },
     "execution_count": 7,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "test.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "id": "2ce1f1da",
   "metadata": {},
   "outputs": [],
   "source": [
    "## Concatination of 2 data frames vertically\n",
    "data = pd.concat([train,test],axis=0)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "id": "ebaaa33d",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "(12026, 4)"
      ]
     },
     "execution_count": 9,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "data.shape"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "id": "6847850b",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "Sentence          0.000000\n",
       "TriggerWord       0.691585\n",
       "TriggerWordLoc    0.691585\n",
       "EventType         0.691585\n",
       "dtype: float64"
      ]
     },
     "execution_count": 10,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "data.isnull().mean()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "id": "4ffd2516",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "array(['Down-regulation;expression;', 'deregulation;', nan, ...,\n",
       "       'expressed;cross-linking;augmented;expression;transcription;up-regulated;activated;transcription;cross-linking;',\n",
       "       'up-regulate;expression;', 'expression;levels;'], dtype=object)"
      ]
     },
     "execution_count": 11,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "data.TriggerWord.unique()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "id": "76ada2c1",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "expression;                     182\n",
       "expressed;                       90\n",
       "binding;                         54\n",
       "binds;                           36\n",
       "bind;                            36\n",
       "induction;                       32\n",
       "overexpression;                  29\n",
       "expression;expression;           28\n",
       "activation;                      28\n",
       "induced;                         26\n",
       "express;                         24\n",
       "activated;                       24\n",
       "bound;                           23\n",
       "regulation;                      22\n",
       "expressing;                      20\n",
       "cross-linking;                   18\n",
       "increase;                        18\n",
       "transcription;                   18\n",
       "increased;                       17\n",
       "regulated;                       17\n",
       "production;                      17\n",
       "Overexpression;                  15\n",
       "induced;expression;              15\n",
       "detected;                        14\n",
       "Expression;                      14\n",
       "interact;                        13\n",
       "produced;                        13\n",
       "interaction;                     13\n",
       "expression;regulated;            12\n",
       "regulation;expression;           12\n",
       "increased;expression;            12\n",
       "interacts;                       11\n",
       "Regulation;expression;           11\n",
       "levels;                          11\n",
       "ligation;                        11\n",
       "inhibited;                       10\n",
       "effect;                          10\n",
       "producing;                       10\n",
       "localization;                    10\n",
       "expression;induced;               9\n",
       "deficient;                        9\n",
       "target;                           8\n",
       "decreased;                        8\n",
       "phosphorylation;degradation;      8\n",
       "present;                          8\n",
       "induce;expression;                8\n",
       "regulate;expression;              8\n",
       "found;                            7\n",
       "regulate;                         7\n",
       "secretion;                        7\n",
       "Name: TriggerWord, dtype: int64"
      ]
     },
     "execution_count": 12,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "data['TriggerWord'].value_counts().iloc[:50]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "id": "478ba24f",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "Gene_expression;                                                428\n",
       "Positive_regulation;                                            378\n",
       "Binding;                                                        310\n",
       "Positive_regulation;Gene_expression;                            178\n",
       "Negative_regulation;                                            152\n",
       "Regulation;                                                     147\n",
       "Regulation;Gene_expression;                                     103\n",
       "Positive_regulation;Positive_regulation;                         96\n",
       "Transcription;                                                   91\n",
       "Gene_expression;Positive_regulation;                             81\n",
       "Gene_expression;Gene_expression;                                 64\n",
       "Negative_regulation;Gene_expression;                             62\n",
       "Negative_regulation;Positive_regulation;                         52\n",
       "Localization;                                                    49\n",
       "Positive_regulation;Transcription;                               48\n",
       "Gene_expression;Negative_regulation;                             37\n",
       "Gene_expression;Regulation;                                      29\n",
       "Regulation;Transcription;                                        28\n",
       "Binding;Binding;                                                 28\n",
       "Positive_regulation;Negative_regulation;                         27\n",
       "Positive_regulation;Localization;                                26\n",
       "Positive_regulation;Positive_regulation;Gene_expression;         25\n",
       "Positive_regulation;Positive_regulation;Positive_regulation;     22\n",
       "Negative_regulation;Positive_regulation;Gene_expression;         22\n",
       "Positive_regulation;Regulation;                                  21\n",
       "Positive_regulation;Gene_expression;Positive_regulation;         21\n",
       "Positive_regulation;Binding;                                     21\n",
       "Transcription;Positive_regulation;                               20\n",
       "Regulation;Positive_regulation;                                  20\n",
       "Regulation;Regulation;                                           18\n",
       "Binding;Positive_regulation;                                     18\n",
       "Negative_regulation;Transcription;                               17\n",
       "Negative_regulation;Binding;                                     16\n",
       "Gene_expression;Binding;                                         13\n",
       "Gene_expression;Positive_regulation;Positive_regulation;         12\n",
       "Phosphorylation;                                                 12\n",
       "Gene_expression;Negative_regulation;Gene_expression;             12\n",
       "Positive_regulation;Gene_expression;Gene_expression;             12\n",
       "Gene_expression;Positive_regulation;Gene_expression;             12\n",
       "Localization;Positive_regulation;                                12\n",
       "Regulation;Binding;                                              12\n",
       "Positive_regulation;Positive_regulation;Transcription;           12\n",
       "Protein_catabolism;                                              11\n",
       "Positive_regulation;Protein_catabolism;                          11\n",
       "Positive_regulation;Phosphorylation;                             11\n",
       "Regulation;Positive_regulation;Gene_expression;                  11\n",
       "Negative_regulation;Negative_regulation;                         11\n",
       "Positive_regulation;Negative_regulation;Positive_regulation;     10\n",
       "Negative_regulation;Localization;                                10\n",
       "Positive_regulation;Gene_expression;Negative_regulation;         10\n",
       "Name: EventType, dtype: int64"
      ]
     },
     "execution_count": 13,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "data['EventType'].value_counts().iloc[:50]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "id": "34043376",
   "metadata": {},
   "outputs": [],
   "source": [
    "data['EventType']=data['EventType'].str.split(';').str[0]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "id": "2c1b0d90",
   "metadata": {},
   "outputs": [],
   "source": [
    "data['EventType'].fillna(data['EventType'].mode()[0],inplace=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "id": "8914eebf",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "Positive_regulation    9488\n",
       "Gene_expression         785\n",
       "Negative_regulation     521\n",
       "Regulation              445\n",
       "Binding                 439\n",
       "Transcription           187\n",
       "Localization             89\n",
       "Phosphorylation          52\n",
       "Protein_catabolism       20\n",
       "Name: EventType, dtype: int64"
      ]
     },
     "execution_count": 16,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "data['EventType'].value_counts()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "id": "f88814d5",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "Sentence             0\n",
       "TriggerWord       8317\n",
       "TriggerWordLoc    8317\n",
       "EventType            0\n",
       "dtype: int64"
      ]
     },
     "execution_count": 18,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "data.isnull().sum()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "id": "2d5931cc",
   "metadata": {},
   "outputs": [],
   "source": [
    "y = data['EventType']"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "id": "639d2dc3",
   "metadata": {},
   "outputs": [],
   "source": [
    "data.reset_index(inplace=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 88,
   "id": "43f6c586",
   "metadata": {},
   "outputs": [],
   "source": [
    "import re\n",
    "from nltk.stem import PorterStemmer\n",
    "from nltk.stem import WordNetLemmatizer\n",
    "from nltk.corpus import stopwords\n",
    "WordNet = WordNetLemmatizer()\n",
    "corpus = []\n",
    "for i in range(0,len(data)):\n",
    "    review = re.sub('[^a-zA-Z]',' ',data['Sentence'][i])\n",
    "    review = review.lower()\n",
    "    review = review.split()\n",
    "    review = [WordNet.lemmatize(word) for word in review if word not in set(stopwords.words('english'))]\n",
    "    review = ' '.join(review)\n",
    "    corpus.append(review)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 89,
   "id": "cd93a9c7",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['regulation interferon regulatory factor gene expression leukemic cell due hypermethylation cpg motif promoter region',\n",
       " 'although bcr abl translocation shown causative genetic aberration chronic myeloid leukemia cml mounting evidence deregulation gene transcription factor interferon regulatory factor irf also implicated pathogenesis cml',\n",
       " 'promoter methylation cpg target site direct deletion insertion gene mechanism reversible permanent silencing gene expression respectively',\n",
       " 'therefore investigated whether irf promoter methylation mutation may involved regulation irf expression leukemia cell',\n",
       " 'whereas promoter mutation structural rearrangement could excluded cause altered irf expression hematopoietic cell irf promoter methylation status found significantly influence irf transcription',\n",
       " 'first treatment irf negative lymphoid myeloid monocytic cell line methylation inhibitor aza deoxycytidine resulted time concentration dependent increase irf mrna protein level',\n",
       " 'second using restriction pcr assay bisulfite sequencing identified specifically methylated cpg site irf negative irf positive cell',\n",
       " 'third clearly determined promoter methylation mechanism irf regulation via reporter gene assay detect association methylational status mrna expression dna methyltransferases methyl cpg binding protein',\n",
       " 'together data suggest cpg site specific irf promoter methylation putative mechanism regulated irf expression leukemia',\n",
       " 'chronic myeloid leukemia cml clonal myeloproliferative disorder typical three phased course chronic accelerated blastic phase reflecting loss differentiation malignant progress inevitably lead death blastic phase',\n",
       " 'hallmark genetic aberration cml reciprocal chromosomal translocation leading expression bcr abl fusion gene aberrant activated tyrosine kinase',\n",
       " 'treatment interferon alpha ifn alpha prolongs survival cml patient associated complete cytogenetic response cml patient',\n",
       " 'recently described impaired expression interferon regulatory factor irf cml correlating poor response ifn alpha treatment',\n",
       " 'cause silencing irf level remained unclear',\n",
       " 'interferon regulatory factor irfs family transcriptional regulator defined characteristic homology dna binding domain',\n",
       " 'play important role regulation various gene ifns interleukin mhc class ii apoptosis differentiation maturation',\n",
       " 'irf icsat pip mum lsirf one member restricted expression pattern predominately b activated lymphocyte irf positive',\n",
       " 'contrast irfs expression irf induced ifns antigen stimulation crosslinking b cell receptor phorbol myristate acetate',\n",
       " 'consistent restriction expression immunocompetent cell mouse deletion irf failed develop mature functionally active b lymphocyte impaired expression irf cml predominately found cell',\n",
       " 'data suggest crucial role irf function immune cell',\n",
       " 'methylation dinucleotide cytosine guanosine motif cpg especially cpg island located promoter region one mechanism gene regulation mammal common event gene silencing human neoplasia',\n",
       " 'opposed normal cell hypermethylation cpg island frequently observed phenomenon every cancer type',\n",
       " 'de novo dna methylation gene cell cycle dna repair apoptosis tumor suppressor gene therefore thought involved tumorigenesis',\n",
       " 'example aberrated gene mgmt dapk p arf p ink b p ink brca cdh apaf',\n",
       " 'cml methylation known regulate expression c abl bcr gene others extent methylation c abl promoter shown associated advanced disease',\n",
       " 'hypermethylation due overexpression dna methyltransferases dnmts remains one possible explanation de novo methylation tumorigenesis',\n",
       " 'recently dnmts shown regulated hematopoietic malignancy',\n",
       " 'methyl cpg binding protein mbps thought inhibit binding transcriptional factor promoter therefore discussed one mechanism transcription inhibition hypermethylation',\n",
       " 'work studied mechanism irf gene expression silencing leukemic cell',\n",
       " 'analyzed irf promoter region genetic aberration methylational status irf positive negative hematopoietic cell',\n",
       " 'cell line',\n",
       " 'k jurkat u obtained atcc american type culture collection rockville usa em lama cml bv sd rpmi dsmz deutsche sammlung von mikroorganismen und zellkulturen gmbh braunschweig germany',\n",
       " 'cell line except bv sd rpmi irf negative',\n",
       " 'cell culture stimulation',\n",
       " 'cell line maintained co rpmi medium glutamine gibco brl eggenstein germany supplemented fetal calf serum gibco brl penicillin streptomycin biochrom berlin germany',\n",
       " 'indicated cell treated aza deoxycytidine azadc azacytidine azac sigma taufkirchen germany different time period',\n",
       " 'owing chemical instability fresh substance added every h',\n",
       " 'sequencing irf promoter',\n",
       " 'analysis irf promoter region permanent aberration insertion deletion mutation pcr amplified two fragment genomic dna extracted depicted cell line commercial kit qiagen hilde germany recommended',\n",
       " 'primer forward ttgagatggagtcttgctctgt reverse ccaggacctcaggaggccagtca forward agcggtgaaactgagagtgcgaggt reverse gccacatcgctgcagtttag',\n",
       " 'product cloned topo ta cloning kit invitrogen groningen netherlands',\n",
       " 'bacterial amplification cloned pcr fragment standard procedure least three clone sample sequenced automated sequencer abi prism applied bio system foster city usa recommended manufacturer',\n",
       " 'expression analysis',\n",
       " 'analyze irf transcriptional level rna extracted cell using commercial rnazol kit paesel frankfurt germany',\n",
       " 'aliquot mug total rna used cdna synthesis described previously',\n",
       " 'rna expression analysis irf reference gene beta actin carried semi quantitative pcr described previously',\n",
       " 'pcr product verified automated sequencing',\n",
       " 'pcr primer condition expression analysis dnmt mbp dnmt dnmt dnmt b mecp mbd mbd mbd published elsewhere',\n",
       " 'analysis irf protein expression standard immunoblotting assay performed described previously',\n",
       " 'briefly protein lysates generated incubating x cell microl ripa buffer np sodiumdesoxycholate sd microg ml phenylmethylsulfonyl fluoride microl ml protease inhibitory mix micromol ml sodiumorthovanadate phosphate buffered saline min ice',\n",
       " 'centrifugation protein concentration supernatant determined bca method pierce rockford il recommended',\n",
       " 'protein lysates microg electrophoresed polyacrylamide gel transferred pvdf membrane immobilon p microm millipore eschborn germany',\n",
       " 'membrane blocked blocking reagent boehringer mannheim germany tbst buffer g l tris hcl g l trisoh g l nacl g l kcl microl l tween h subsequently incubated primary antibody indicated horseradish peroxidase conjugated secondary antibody anti mouse anti goat igg dako hamburg germany respectively',\n",
       " 'membrane developed ecl detection kit amersham pharmacia biotech freiburg germany',\n",
       " 'primary antibody goat anti irf icsat santa cruz biotechnology santa cruz ca mouse anti beta actin ac sigma',\n",
       " 'methylation specific restriction pcr assay',\n",
       " 'dna extracted commercial kit qiagen recommended',\n",
       " 'since restriction ability several endonuclease inhibited methylation target sequence used methylation sensitive enzyme hpaii haeii isochizomer bsp ii bsh mbi fermentas st leon rot germany',\n",
       " 'control methylation resistant enzyme mspi enzyme recognition site target promoter ecori used',\n",
       " 'dna microg digested u respective enzyme h ensure complete cleavage additional u h',\n",
       " 'thereafter ng digested dna used pcr amplification two fragment f f spanning part irf promoter genbank u see figure',\n",
       " 'sequence primer f forward ttgagatggagtcttgctctgt f reverse atcacttccagacttcagttcacct bp f forward aaggtgaactgaagtctggaagtga f reverse ccaggacctcaggaggccagtca bp',\n",
       " 'pcr condition described elsewhere',\n",
       " 'pcr performed annealing temperature degreesc cycle',\n",
       " 'dna methylated specific site sensitive enzyme able digest dna amplification took place case methylation dna digested product generated',\n",
       " 'pcr product electrophoresed agarose gel stained ethidium bromide photographed',\n",
       " 'pcr product verified automated sequencing',\n",
       " 'bisulfite treatment',\n",
       " 'dna extracted described',\n",
       " 'bisulfite treatment dna leading conversion unmethylated cytosine uracil residue change methylated cytosine residue performed described follows',\n",
       " 'briefly microg dna microg poly da dt poly da dt copolymer amersham pharmacia biotech denaturated min degreesc naoh volume microl',\n",
       " 'fresh solution microl mm hydrochinon sigma microl sodium bisulfite ph sigma added solution gently mixed overlayed mineral oil incubated dark h degreesc',\n",
       " 'aqueous phase recovered using wizard dna clean system promega mannheim germany',\n",
       " 'purified dna subsequently mixed naoh final concentration incubated min degreesc ensure complete desulfonisation',\n",
       " 'dna ethanol precipitated presence vol sodium acetate washed ethanol resuspended microl h',\n",
       " 'subsequent pcr amplification microl bisulfite treated dna used cloning two fragment irf promoter b b ii pcr vector topo ta cloning kit invitrogen see figure',\n",
       " 'primer used pcr amplification b b ii fragment contain putative altered sequence sense strand due bisulfite treatment converted cytosine residue written bold letter b forward tatttggatttttagggagtttttttt b reverse acccaactcccttaaactattaaact bp b ii forward agtttaatagtttaagggagttgggt b ii reverse ctcaccctaaactcaaaactaaaaac bp',\n",
       " 'bacterial amplification cloned pcr fragment standard procedure eight clone sample sequenced automated sequencer abi prism applied biosystems',\n",
       " 'vitro methylation reporter gene assay',\n",
       " 'irf promoter reporter gene construct generously provided j hiscott',\n",
       " 'construct methylated vitro cpg methylase ss recommended manufacturer ne biolabs complete methylation checked via restriction analysis figure',\n",
       " 'reporter gene assay using dual luciferase assay promega performed similar previous report',\n",
       " 'briefly nm reporter construct transfection control construct expressing renilla luciferase gene transiently co expressed via electroporation',\n",
       " 'control construct served internal reference transfection efficiency',\n",
       " 'forty eight hour transfection luciferase activity measured lb p microlumat eg g berthold bad wildbad germany',\n",
       " 'irf promoter activation quantified ratio measured firefly light unit flu relative renilla rlu',\n",
       " 'experiment carried least three time',\n",
       " 'absence irf expression leukemia cell due promoter alteration',\n",
       " 'previously demonstrated lack irf expression leukemia patient specifically cml cell',\n",
       " 'demonstrate absence irf expression various hematopoietic cell line jurkat cell leukemia cml bcr abl positive cell line k bcr abl positve erythroleukemia u monocytic leukemia em lama bcr abl positve myeloid leukemia sd bcr abl positive acute lymphoblastic leukemia pre b rpmi multiple myeloma bv bcr abl positive b cell line figure',\n",
       " 'sequencing irf promoter could excluded absence irf expression cell line due genetic aberration',\n",
       " 'however bp change nucleotide c c could detected irf positive bv irf negative lama em k figure b',\n",
       " 'position c substitution found em k cml whereas jurkat bv sd exhibited mixed c sequence u lama rpmi substitution figure b',\n",
       " 'consequently alteration unlikely affect irf expression',\n",
       " 'increase irf expression hematopoietic cell demethylating treatment',\n",
       " 'next analyzed whether promoter methylation could responsible regulation irf expression',\n",
       " 'region including exon irf promoter exhibited large number cpg rich sequence figure',\n",
       " 'several chemical substance aza deoxycytidine azadc azacytidine azac inhibit de novo maintenance methylation thus used discern promoter methylation',\n",
       " 'used azadc generate unmethylated dna',\n",
       " 'h azadc treatment resulted concentration dependent activation irf transcription jurkat cml cell well u k em cell line figure',\n",
       " 'irf transcription induced time dependent manner observed early h treatment azadc increased time h figure b',\n",
       " 'time strength appearance irf transcript varied among cell line e cml responded strongest azadc treatment data shown',\n",
       " 'line azadc treatment cml lama cell also translated induction irf protein expression figure c',\n",
       " 'accordingly treatment irf positive cell line bv sd rpmi azadc effect irf expression figure',\n",
       " 'difference effect azac versus azadc increased irf mrna level cml cell well data shown',\n",
       " 'implied promoter methylation may control irf expression alternative explanation may activation positive transcriptional regulator irf azadc azac',\n",
       " 'methylation sensitive enzyme cut specific site irf promoter hematopoietic cell',\n",
       " 'investigate promoter methylation regulatory mechanism irf gene expression restriction pcr assay performed methylated dna would cut enabling subsequent pcr amplification vice versa',\n",
       " 'genomic dna leukemic cell jurkat cml u k em bv digested methylation sensitive enzyme hpaii bsh haeii isochizomer bsp ii',\n",
       " 'ecori recognition site within irf promoter methylation resistant enzyme mspi served control',\n",
       " 'two separate amplification reaction performed generating two fragment f f figure',\n",
       " 'digestion hpaii bsp ii sufficient pcr amplification f f detected dna irf negative jurkat cml u k em cell suggesting promoter methylation restriction protection respective recognition site figure b c',\n",
       " 'notably irf positive sd cell digestion methylation sensitive enzyme completely inhibited amplification f f',\n",
       " 'irf positive bv cell hpaii bsh digestion significantly reduced amplifiable dna message f figure c whereas amplification f affected figure b',\n",
       " 'implied irf transcription sd bv cell associated le promoter methylation bv especially hpaii site compared tested irf negative cell',\n",
       " 'specific cpg site irf promoter methylated hematopoietic cell',\n",
       " 'order exactly map methylation site within irf promoter treated dna jurkat cml u k em cell well sd rpmi bv control cell bisulfite chemically convert unmethylated cytosine uracil whereas effect methylated cytosine e cpg',\n",
       " 'technique especially useful detection unknown methylation pattern',\n",
       " 'pcr amplification cloning sequencing bisulfite treated dna showed specific methylation pattern analyzed cpg site cell line figure table',\n",
       " 'general methylational status ranged one cell line nearly non methylated irf promoter sd irf positive completely methylated irf promoter cml irf negative',\n",
       " 'interestingly percentage cpg methylation irf promoter irf positive cell low mean compared irf negative cell mean figure table',\n",
       " 'region r hypermethylated cpg site mean number methylated clone methylated cpgs found cell except sd rpmi region r hypomethylated cpg site mean number methylated clone methylated cpgs found cell except cml u figure table',\n",
       " 'intriguingly stretch cpg site r detected region highly methylated irf negative mean number methylated clone methylated cpgs totally non methylated irf positive cell figure b',\n",
       " 'furthermore three cpg site end two cpg motif end showed direct correlation high methylation status absence irf expression',\n",
       " 'addition two cpg site located nfkappab sp element le methylated irf positive irf negative cell mean number methylated clone versus',\n",
       " 'result indicate involvement cpg methylation regulation irf expression leukemic cell',\n",
       " 'vitro methylation irf promoter reporter construct decrease activity',\n",
       " 'provide evidence direct effect methylational status irf promoter activity performed reporter gene assay irf promoter construct vitro methylation',\n",
       " 'complete methylation construct checked via restriction assay methylation sensitive endonuclease figure',\n",
       " 'intriguingly methylation irf promoter significantly decreased promoter activity irf positive sd cell figure b',\n",
       " 'silencing effect cpg methylation restricted irf positive cell since vitro methylation led abrogation promoter activity irf negative jurkat cell figure c',\n",
       " 'contrast control methylation reporter construct different promoter fasl well empty vector effect reporter activity data shown',\n",
       " 'data proved direct association methylation activity irf promoter',\n",
       " 'mrna expression dna methyltransferases methyl cpg binding protein may associated irf promoter methylation',\n",
       " 'since abundance dnmt mbp contribute promoter regulation via methylation studied mrna expression investigate possible mechanism observed methylation difference irf promoter',\n",
       " 'end detect significant difference dnmt dnmt dnmt dnmt b mbp mbd mbd mbd mecp mrna expression irf positive negative cell figure',\n",
       " 'fact analyzed cell moderate high mrna level tested dnmt mbps difference expression correlated irf status',\n",
       " 'result indicate distinct cause methylation difference irf positive negative cell rather change dnmt mbp mrna transcription',\n",
       " 'many genetic lesion known influence gene expression tumor suppressor gene',\n",
       " 'whereas mutation deletion insertion permanent effect reversible mechanism gene methylation expression activation transcription factor respectively',\n",
       " 'studied putative cause absent irf expression leukemia cell first focused genetic aberration promoter',\n",
       " 'observed genetic alteration irf promoter account lack irf expression detected base pair change position c substitution position c substitution position c substitution unlikely responsible absent irf expression since first two mutation found irf positive negative cell whereas latter change detected consistently irf negative positive cell may thus polymorphism',\n",
       " 'three substitution change known putative transcription factor binding site also affect restriction site primer binding site used assay',\n",
       " 'however permanent genetic variation irf coding sequence deletion mutation resulting stop codon excluded sequence analysis',\n",
       " 'since irf expression cell line cml induced demethylation successful ifn alpha therapy respectively existence genetic aberration seems unlikely',\n",
       " 'investigated whether previously described regulation irf expression human myeloid leukemia due differential hypermethylation promoter since presented expression due azadc treatment might also result activation positive transcriptional regulator irf',\n",
       " 'methylation cpg site common mechanism silencing gene leukemia also shown another irf irf pu interacting partner irf',\n",
       " 'elucidate relevance mechanism regulation irf expression various leukemic cell treated demethylating agent promoter sequenced bisulfite treatment',\n",
       " 'found irf expression could indeed connected methylation status distinct cpg motif irf promoter',\n",
       " 'figure cpg site shown bottom line whose hypermethylation may account absence irf expression respective cell',\n",
       " 'one adjacent identified regulatory element nfkappab site indicating possible involvement site',\n",
       " 'two cpg site methylation status irf positive lower irf negative cell',\n",
       " 'cpg site located nfkappab sp element thus may also play role regulation irf expression',\n",
       " 'shown nfkappab element play important role irf induction irf expression depends binding transactivator c rel element irf promoter',\n",
       " 'furthermore methylation central cpg nfkappab element inhibits binding nfkappab protein complex promoting significance observed methylation difference irf positive negative cell',\n",
       " 'via vitro methylation reporter gene assay could clearly appoint silencing irf promoter methylation effect may thus mechanism irf deregulation vivo',\n",
       " 'one possible cause aberrant methylation tumorigenesis increased level dnmts pathogenetic process',\n",
       " 'colon lung hematologic malignancy overexpression dnmt maintenance dnmt detected',\n",
       " 'furthermore shown cml cell acute phase exhibited elevated level three known dnmts cml cell chronic phase expressed normal level dnmts compared normal bone marrow cell',\n",
       " 'interestingly positive correlation dnmt expression level hypermethylation p ink b detected aml',\n",
       " 'work detect significant mrna expression difference selected dnmt mbp making unlikely cause observed methylation thus irf expression difference leukemia cell',\n",
       " 'finding irf expression silenced promoter hypermethylation might represent mechanism account previously observed loss irf expression cml',\n",
       " 'indeed several clinical trial leukemia patient patient myelodysplastic syndrome demonstrated potential clinical benefit treatment demethylating agent',\n",
       " 'expression another irf ifn consensus sequence binding protein icsbp irf impaired myeloid leukemia especially cml',\n",
       " 'contrast irf loss irf could reverted icsbp negative cell line em cml k lama treatment azadc figure azadc effect icsbp level icsbp positive u cell figure',\n",
       " 'data suggest distinct regulatory mechanism two irfs',\n",
       " 'irf similar many classical tumor suppressor gene p ink b p ink p may thus subject alteration promoter methylation status leading expression change might contribute initiation progression cancer',\n",
       " 'still obvious functional diversity irf remains remarkable fully explained irf promoter methylation status',\n",
       " 'example irf primarily known oncogenic feature',\n",
       " 'multiple myeloma mm translocation chromosome q reported lead fusion gene immunoglobulin heavy chain igh irf resulting subsequent overexpression irf',\n",
       " 'addition abundant irf expression found marker various subset lymphoma diffuse large b cell lymphoma primary effusion lymphoma marginal zone lymphoma adult cell leukemia',\n",
       " 'draw complex picture role irf',\n",
       " 'regulation irf may promote leukemogenesis myeloid cell context recently confirmed irf icsbp double knock mouse irf regulation may induce growth advantage lymphoma mm',\n",
       " 'taken together data suggest irf promoter methylation regulates irf expression aberrant expression irf certain type leukemia may consequence irf promoter hypermethylation',\n",
       " 'foxp represses retroviral transcription targeting nf kappab creb pathway',\n",
       " 'forkhead box fox winged helix transcription factor regulate multiple aspect immune responsiveness foxp recognized essential functional marker regulatory cell',\n",
       " 'herein describe downstream signaling pathway targeted foxp may negatively impact retroviral pathogenesis',\n",
       " 'overexpression foxp hek purified cd cell resulted dose dependent time dependent decrease basal level nuclear factor kappab nf kappab activation',\n",
       " 'deletion carboxyl terminal forkhead fkh domain critical nuclear localization dna binding activity abrogated ability foxp suppress nf kappab activity hek cell jurkat primary human cd cell',\n",
       " 'demonstrate foxp suppressed transcription two human retroviral promoter hiv human cell lymphotropic virus type htlv utilizing nf kappab dependent nf kappab independent mechanism',\n",
       " 'examination latter identified camp responsive element binding protein creb pathway target foxp',\n",
       " 'finally comparison percent foxp cd cd cell htlv proviral load htlv infected asymptomatic carrier patient htlv associated myelopathy tropical spastic paraparesis suggested high foxp expression associated low proviral load absence disease',\n",
       " 'result suggest expanded role foxp regulating nf kappab creb dependent cellular viral gene expression',\n",
       " 'past several year mounting evidence shown immune tolerance healthy individual maintained population lymphocyte known regulatory cell tregs',\n",
       " 'component system protein known foxp shown absolutely required development function tregs',\n",
       " 'foxp play important role maintaining immune tolerance blocking cell proliferation production inflammatory protein known cytokine little known molecular mechanism used foxp accomplish event',\n",
       " 'present study expands understanding foxp maintains check inappropriate immune response demonstrating foxp block activation key inducible protein nuclear factor kappab nf kappab camp responsive element binding protein creb',\n",
       " 'since nf kappab creb integrally involved controlling cell cycle progression inflammatory cytokine production replication numerous virus level transcription understanding mechanism foxp function regulate cellular viral gene expression may aid discovery therapeutic approach designed rescue expression function foxp found deficient several autoimmune disease virus induced disorder',\n",
       " 'immunological tolerance self antigen result deletion self reactive lymphocyte thymus central tolerance suppression activation potentially self reactive lymphocyte periphery peripheral tolerance',\n",
       " 'suppression pathogenic cell response mediated naturally arising cd cd regulatory cell tregs',\n",
       " 'deficiency treg development function linked severe autoimmune disorder known immune dysregulation polyendocrinopathy enteropathy x linked syndrome ipex',\n",
       " 'addition recent study provided strong evidence dysregulation treg development function may significant factor pathogenesis several autoimmune disorder e g multiple sclerosis myasthenia gravis type diabetes virus induced immunologic disorder e g human lymphotropic virus type htlv associated myelopathy tropical spastic paraparesis ham tsp hiv induced aid',\n",
       " 'transcription factor foxp amino acid kda protein expressed high level cd cd hi cell previously shown absolutely critical treg development function',\n",
       " 'foxp contains proline rich amino terminal domain reported function nuclear factor activated cell nf nuclear factor kappab nf kappab binding domain central region containing zinc finger leucine zipper potentially important protein protein interaction carboxyl terminal forkhead fkh domain required nuclear localization dna binding activity',\n",
       " 'functional inactivation foxp genetic mutation affecting foxp coding region demonstrated ipex repression foxp expression htlv encoded transactivator protein tax recently reported patient ham tsp result loss regulatory activity cd cd hi cell',\n",
       " 'although clear foxp regulates cell proliferation cytokine production little known concerning molecular mechanism foxp function',\n",
       " 'first evidence indicate foxp promotes development function regulatory cell came report ziegler colleague suggested foxp could inhibit transcriptional activation physically interacting forkhead binding site located immediately adjacent critical ci acting nf binding site found various cytokine promoter e g il promoter',\n",
       " 'study also demonstrated foxp could repress activation synthetic reporter vector containing sv promoter three tandem copy forkhead binding site',\n",
       " 'result provided additional evidence suggesting foxp transcriptional repression mediated binding sequence specific manner promoter containing forkhead binding site',\n",
       " 'recent study bettelli colleague demonstrated foxp could inhibit nf well nf kappab activation although mechanism suppression shown involve direct protein protein interaction nf nf kappab foxp rather binding foxp promoter element adjacent ci acting nf nf kappab site',\n",
       " 'collectively data suggested foxp may function transcriptional repressor potentially formation dna protein protein protein interaction',\n",
       " 'present study expanded upon observation defining additional requirement foxp mediated repression nf kappab activation investigated whether foxp could target additional signaling pathway examining transcriptional activation nf kappab dependent nf kappab independent retroviral pathogen',\n",
       " 'characterization molecular target foxp mechanism utilized foxp support treg development function aid understanding role tregs play pathogenesis human autoimmune disease',\n",
       " 'foxp suppresses nf kappab dependent transcriptional activation',\n",
       " 'ascertain molecular mechanism foxp function promote regulatory function cd cd hi cell first confirmed function foxp repressor activation nf kappab previously implicated target forkhead winged helix family transcription factor e g foxj foxo',\n",
       " 'analyzed effect foxp overexpression nf kappab activation hek cell dose response time course analysis',\n",
       " 'transfection hek cell nf kappab luciferase reporter vector presence absence increasing concentration foxp expression vector control vector enhanced green fluorescent protein egfp performed cell harvested h assay luciferase activity foxp mrna expression',\n",
       " 'result indicated concentration foxp transfected cell increase ng level nf kappab activation decrease proportionally figure',\n",
       " 'foxp mrna also assayed monitor activity foxp expression vector figure b',\n",
       " 'since nf kappab activation partially affected transfection high concentration control vector determined fold inhibition nf kappab activation foxp compared control vector concentration figure',\n",
       " 'fold inhibition nf kappab activation directly proportional level foxp mrna expression detected real time rt pcr',\n",
       " 'determine level foxp mediated suppression nf kappab activation time hek cell transfected nf kappab luciferase reporter vector expression vector encoding foxp egfp control vector harvested',\n",
       " 'shown figure c nf kappab activation suppressed overexpression foxp time point',\n",
       " 'extending result established vitro hek cell line primary human lymphocyte overexpression foxp purified cd cell three healthy donor also regulated steady state level nf kappab activation figure',\n",
       " 'result recapitulate bettelli colleague demonstrating foxp function part block nf kappab dependent transcription human cell line well primary human cd cell',\n",
       " 'carboxyl terminal fkh domain required suppression nf kappab activation cell',\n",
       " 'define requirement foxp respect inhibition nf kappab dependent transcription utilized mutant foxp lacking fkh domain figure similar scurfy mutant foxp mouse mutant foxp protein patient ipex',\n",
       " 'unlike full length foxp localizes almost exclusively nucleus bind sequence specific manner forkhead binding site deltafkh mutant fails localize nucleus thus interact promoter element nuclear protein',\n",
       " 'therefore utilized deltafkh mutant determine whether nuclear localization function associated fkh domain foxp prerequisite inhibition nf kappab activation',\n",
       " 'although foxp interaction nf kappab presumably take place nucleus may also possible cytoplasmic foxp protein bind nf kappab cytoplasm prevent localization nucleus following activation stimulus',\n",
       " 'overexpression full length foxp deltafkh able suppress activation cotransfected nf kappab reporter vector hek cell figure b',\n",
       " 'foxp deltafkh expressed high level following transfection detected real time rt pcr unpublished data',\n",
       " 'data appear suggest carboxyl terminal fkh domain critically important foxp regulate nf kappab dependent transcription',\n",
       " 'however nf kappab activation blocked similar extent full length foxp deltafkh jurkat cell figure c primary human cd cell figure',\n",
       " 'western blot analysis nf kappab p expression demonstrated foxp deltafkh block nf kappab activation level p protein expression figure e',\n",
       " 'result interesting respect foxp function suggest carboxyl terminal fkh domain possibly nuclear localization dispensable foxp function cell population',\n",
       " 'alternative interpretation may include possibility localization deltafkh differ epithelial cell cell',\n",
       " 'either case result suggest cell type specific mechanism action foxp mutant',\n",
       " 'foxp suppresses hiv gene expression part blocking activation nf kappab',\n",
       " 'foxp function repressor nf kappab dependent gene expression hypothesized foxp overexpression could selectively regulate transcription promoter previously shown responsive nf kappab',\n",
       " 'address question examined transcriptional activation hiv ltr contains two tandem ci acting nf kappab binding site located position respect transcription initiation site',\n",
       " 'nf kappab play crucial role regulating gene expression directed hiv ltr cd cell',\n",
       " 'overexpression full length foxp deltafkh hek cell able inhibit basal activation hiv ltr figure similar previously demonstrated synthetic nf kappab reporter vector figure b',\n",
       " 'furthermore hiv ltr activation suppressed full length foxp deltafkh jurkat cell figure b',\n",
       " 'demonstrate foxp mediated hiv ltr repression associated interaction nf kappab bound hiv ltr compared basal activation hiv ltr identical hiv ltr lacking nf kappab site located hiv delta kappab ltr figure c',\n",
       " 'mutant hiv ltr construct exhibited reduced level transcription compared parental hiv ltr purified healthy donor cd cell unpublished data',\n",
       " 'however directly comparing effect foxp overexpression activation two viral promoter demonstrated foxp capable suppressing transcriptional activation hiv ltr figure compared mutated hiv ltr figure e',\n",
       " 'result suggest foxp regulation hiv ltr activation mediated least part ci acting nf kappab binding site',\n",
       " 'residual level inhibition hiv delta kappab ltr foxp may due nf binding site located upstream nf kappab site within hiv ltr',\n",
       " 'transactivation function htlv tax suppressed foxp',\n",
       " 'previous study bettelli colleague demonstrated foxp repress basal level nf kappab activation well tumor necrosis factor alpha stimulated nf kappab activation',\n",
       " 'next step determine whether foxp could also suppress activation nf kappab caused strong viral transactivator protein',\n",
       " 'htlv encodes multifunctional transactivator protein tax capable activating nf kappab creb pathway',\n",
       " 'since htlv tax protein function multiple level cytoplasm nucleus stimulate activation nf kappab hypothesized overexpression foxp may interfere process',\n",
       " 'however since tax dependent htlv gene expression independent nf kappab also hypothesized foxp would affect tax dependent activation htlv ltr',\n",
       " 'test hypothesis overexpressed htlv tax full length foxp deltafkh hek cell cotransfected htlv ltr nf kappab reporter vector',\n",
       " 'shown figure htlv tax strongly regulated nf kappab dependent transcriptional activation fold',\n",
       " 'interestingly overexpression foxp deltafkh suppressed tax mediated activation nf kappab dependent transcription',\n",
       " 'observation suggest carboxyl terminal fkh domain required inhibiting activation nf kappab presence tax hek cell strikingly similar requirement foxp inhibition basal nf kappab activation shown figure b',\n",
       " 'transactivation htlv ltr stimulated fold overexpression tax figure b transfection foxp suppressed tax dependent htlv ltr activation although htlv ltr activation presence absence tax independent nf kappab nf another transcriptional activator known interact foxp',\n",
       " 'furthermore overexpression deltafkh also led suppression htlv transactivation tax similar extent full length foxp figure b',\n",
       " 'suppressive effect shown figure b result foxp regulating expression transfected tax plasmid determined real time rt pcr figure c',\n",
       " 'result strongly suggest foxp interacts transcriptional regulator addition nf kappab nf carboxyl terminal fkh domain therefore localization nucleus required inhibition tax mediated htlv ltr activation even hek cell',\n",
       " 'determine whether foxp inhibited transactivation function tax directly associating viral protein generated expression vector htlv tax fused frame carboxyl terminus gal dna binding domain gal bd',\n",
       " 'gal bd tax fusion protein activated transcription synthetic promoter containing five gal binding site gal bd insufficient stimulate transcription figure',\n",
       " 'transactivation gal resposive promoter gal bd tax remained relatively unaffected overexpression either egfp control foxp deltafkh suggesting foxp repress tax transactivation directly interfacing htlv tax protein',\n",
       " 'confirm foxp direct effect htlv replication transfected hek cell well characterized htlv infectious molecular clone termed ach presence full length foxp deltafkh control vector',\n",
       " 'ach previously shown direct expression viral antigen produce infectious virus transform cd cell vitro vivo',\n",
       " 'h amount viral antigen expression case tax mrna detected sensitive real time rt pcr assay',\n",
       " 'illustrated figure level tax mrna synthesized ach regulated presence foxp compared level produced presence control vector',\n",
       " 'deltafkh discernable affect tax expression',\n",
       " 'data indicate foxp capable repressing expression tax infectious htlv molecular clone',\n",
       " 'increased foxp protein expression associated low htlv proviral load',\n",
       " 'since foxp expressed almost exclusively within cd cd cell major viral reservoir htlv important determine whether association foxp htlv replication infected patient',\n",
       " 'therefore quantitated foxp protein expression cd cd cell flow cytometry htlv proviral load surrogate marker viral replication real time pcr eight patient ham tsp eight asymptomatic carrier ac',\n",
       " 'data analysis summarized table',\n",
       " 'expected patient ham tsp exhibited significantly higher proviral load indicated htlv proviral dna copy cell compared ac p',\n",
       " 'percentage foxp cell within cd cd cell population significantly greater ac ham tsp patient p',\n",
       " 'data suggest high level foxp protein expression associated reduced htlv replication vivo',\n",
       " 'also support recent report high proviral load shown correlate high tax mrna htlv infected patient associated reduced foxp expression',\n",
       " 'creb target transcriptional repression foxp',\n",
       " 'although foxp could regulate tax dependent transactivation htlv ltr figure b inhibit tax expression infectious molecular clone figure foxp failed modulate tax function absence viral promoter figure',\n",
       " 'result led u hypothesize foxp act htlv gene expression interacting protein important driving htlv ltr activity vivo',\n",
       " 'previous study demonstrated tax responsive element within htlv ltr play crucial role driving tax mediated transactivation htlv ltr',\n",
       " 'tax responsive element shown resemble creb binding site bind creb vitro vivo facilitate htlv ltr activation presence absence tax',\n",
       " 'ching colleague demonstrated addition dominant negative creb expression vector resulted nearly complete inhibition tax mediated activation htlv ltr blocking nf kappab activation addition dominant negative ikkbeta expression vector effect tax transactivation htlv ltr',\n",
       " 'therefore hypothesized foxp may inhibit tax transactivation htlv ltr via disruption creb signaling pathway',\n",
       " 'test possibility hek cell transfected htlv ltr synthetic creb reporter vector along control expression vector egfp expression vector encoding foxp deltafkh',\n",
       " 'shown figure foxp regulated basal activation htlv ltr transcription synthetic creb reporter vector suggesting foxp regulates htlv ltr activation targeting creb pathway',\n",
       " 'deletion fkh domain foxp dampened suppressive effect foxp completely abrogate suppression seen nf kappab responsive promoter hek cell',\n",
       " 'like nf kappab activation creb transcriptional activation also suppressed expression foxp similar extent deltafkh healthy donor cd cell figure b',\n",
       " 'similarly foxp deltafkh also repressed basal htlv ltr activation primary human cd cell figure c',\n",
       " 'knowledge first evidence implicating creb molecular target foxp',\n",
       " 'observed nf kappab activation deltafkh potent inhibitor creb activation cd cell hek cell indicating cell type specific mechanism action may govern function foxp mutant',\n",
       " 'determine whether foxp functioned directly signaling creb utilized expression vector encoding creb c jun member activator protein family transcription factor fused frame gal bd gal bd creb gal bd c jun',\n",
       " 'shown figure activation gal responsive reporter vector gal bd creb regulated foxp compared control vector egfp indicating foxp function directly indirectly interacting creb',\n",
       " 'however foxp failed markedly affect transcriptional activation gal bd c jun c jun also demonstrated bind htlv ltr gal bd tax see figure',\n",
       " 'importantly mechanism foxp mediated inhibition creb dependent transcription due block creb protein expression determined western blot analysis figure e',\n",
       " 'although result demonstrate foxp function co repressor creb activation addition nf kappab nf unable detect direct physical interaction creb foxp coimmunoprecipitation mammalian two hybrid analysis unpublished data',\n",
       " 'therefore data suggest foxp may interfere creb signaling upstream event phosphorylation creb recruitment function coactivator protein creb binding protein cbp p',\n",
       " 'foxp antagonizes creb transcriptional activation disrupting coactivator recruitment',\n",
       " 'stimulation creb dependent transcription reagent activate adenylate cyclase increase camp level e g forskolin increase transactivation potential creb phosphorylation serine protein kinase permit binding recruitment coactivators cbp p promoter',\n",
       " 'phosphorylation serine however affect dna binding activity creb case',\n",
       " 'addition forskolin hek cell stimulated activation creb reporter vector fold figure',\n",
       " 'overexpression foxp capable regulating forskolin induced creb transcriptional activation',\n",
       " 'functional interaction foxp creb affect dna binding activity creb show modest decrease activating transcription factor atf dna binding activity presence forskolin determined transcription factor elisa figure b',\n",
       " 'foxp shown bind repress activation nf kappab nf exactly foxp function bring affect elucidated',\n",
       " 'determine foxp block creb dependent transcription examined whether foxp capable disrupting recruitment coactivator protein preventing phosphorylation creb serine prerequisite coactivator recruitment',\n",
       " 'since event required creb dependent gene expression hypothesized foxp may affect creb activation step',\n",
       " 'determine whether foxp disrupt function recruitment coactivator protein p introduced gal reporter vector gal bd creb expression vector hek cell absence presence foxp p control expression vector figure c',\n",
       " 'expected p overexpression stimulated transcription gal bd creb fusion protein',\n",
       " 'foxp repressed basal level gal bd creb activation fold effectively neutralizing gal bd creb activation presence p',\n",
       " 'next analyzed effect foxp phosphorylation creb serine forskolin treated hek cell western blot analysis',\n",
       " 'overexpression foxp failed reduce detectable level creb phosphorylation using phosphospecific antibody creb unpublished data',\n",
       " 'however attempted determine whether foxp could physically interact coactivator protein p found p immunoprecipitated foxp protein overexpressed hek cell figure',\n",
       " 'collectively result suggest foxp antagonizes creb dependent gene expression directly interacting coactivator p interfering function recruitment creb responsive promoter sequence',\n",
       " 'present study show foxp function potent repressor nf kappab creb dependent transcriptional activation',\n",
       " 'furthermore carboxyl terminal fkh domain appears dispensable mediating effect least cell population',\n",
       " 'observation may become important light recent report suggesting foxp expression thymic epithelial cell crucial directing development cell thymus',\n",
       " 'interestingly majority genetic mutation associated ipex severe autoimmune disorder caused functional inactivation foxp map carboxyl terminal fkh domain leucine zipper domain central region protein',\n",
       " 'one mutation associated ipex date mapped amino terminal proline rich region',\n",
       " 'possible fkh domain complex tertiary structure particularly sensitive misfolding caused genetic mutation intact fkh domain absolutely critical promoting foxp function nucleus whereas structure amino terminal proline rich region may tolerate certain mutation long nf kappab nf binding motif remains unaltered',\n",
       " 'motif may also include zinc finger domain',\n",
       " 'logical region may targeted amino terminal proline rich region foxp rel homology domain found nf kappab nf family protein',\n",
       " 'region may also important respect foxp function leucine zipper domain demonstrated number mutation associated ipex mapped region foxp',\n",
       " 'role domain foxp function remains uncharacterized may play role dimer formation foxp family member',\n",
       " 'pathogenesis number retroviral induced immunologic disorder hiv aid htlv ham tsp associated dysregulation foxp expression also examined role foxp retroviral gene expression',\n",
       " 'hiv ltr activation cd cell critically dependent two tandem nf kappab site located nucleotide position within hiv enhancer region whereas htlv ltr activation presence absence htlv encoded transactivator protein tax independent nf kappab',\n",
       " 'knowledge first time foxp shown direct effect hiv ltr transcription',\n",
       " 'deletion nf kappab site within hiv enhancer region reduced responsiveness hiv ltr foxp mediated suppression',\n",
       " 'addition fkh domain foxp required inhibitory effect hek cell jurkat cell similar foxp mediated suppression synthetic nf kappab reporter',\n",
       " 'direct effect foxp regulating hiv gene expression correlate well recently reported evidence indicating higher regulatory activity cd cd cell hiv infected patient associated lower hiv viral load patient',\n",
       " 'foxp also affected two well known function htlv tax transactivation nf kappab pathway surprisingly transactivation htlv ltr',\n",
       " 'transactivation htlv ltr tax involves interaction atf creb factor tax nucleus',\n",
       " 'binding tax enhances atf creb dimerization promotes assembly tax atf creb complex onto specific sequence viral promoter known tax responsive element',\n",
       " 'series step allows tax recruit coactivator protein cbp p viral promoter facilitate high level viral gene expression',\n",
       " 'transactivation nf kappab pathway tax inhibited overexpression full length foxp deltafkh seen basal activation hiv ltr synthetic nf kappab reporter hek cell',\n",
       " 'however tax mediated transactivation htlv ltr inhibited overexpression full length foxp well deltafkh hek cell cd cell',\n",
       " 'demonstrated foxp directly affect functioning tax rather foxp targeted transcription factor required tax transactivation e nf kappab unknown cellular factor identified study creb',\n",
       " 'negative effect foxp htlv gene expression confirmed utilizing htlv infectious molecular clone',\n",
       " 'importantly demonstrated htlv infected individual highest level foxp protein expression within cd cd cell population exhibited lower proviral load individual lowest level foxp protein expression',\n",
       " 'previous study demonstrated htlv proviral load directly correlate htlv tax mrna load frequency immunopathogenic virus specific cd cell disease severity patient ham tsp',\n",
       " 'result important implication utility foxp controlling viral gene expression thus pathogenesis ham tsp',\n",
       " 'therefore foxp becomes attractive target development novel therapeutic application directed modulating expression important regulatory protein especially light recent observation expression foxp also regulated htlv tax',\n",
       " 'activation htlv ltr depends primarily atf creb protein whether presence absence tax investigated whether foxp could interact additional cellular signaling pathway',\n",
       " 'dna binding activity creb case constitutive transactivation potential creb regulated phosphorylation creb recruitment cbp p',\n",
       " 'data demonstrate foxp interferes latter two process recruitment coactivator protein p resulting transcriptional activation blocked foxp',\n",
       " 'may result physical interaction detected foxp p',\n",
       " 'respect htlv ltr activity full length foxp inhibited basal tax dependent transcription deltafkh appeared le effective suppressing basal activation inhibition compared tax dependent activation inhibition',\n",
       " 'effect deltafkh basal activation htlv ltr hek cell similar shown synthetic creb reporter suggesting fkh domain foxp important level',\n",
       " 'observed nf kappab activation foxp mutant lacking fkh domain stronger inhibitor creb activation cd cell hek epithelial cell',\n",
       " 'therefore appears cd cell fkh domain dispensable proper functioning foxp respect nf kappab creb activation',\n",
       " 'summary knowledge first direct evidence implicating role treg specific transcription factor foxp regulating retroviral gene expression',\n",
       " 'addition identify creb pathway molecular target foxp',\n",
       " 'since creb shown regulate multiple gene involved transcription e g jund c fo signal transducer activated cell stat cell cycle e g p ink b cyclin cyclin immune regulation e g il il cell receptor alpha reviewed finding presented report broaden potential range signaling pathway control regulatory protein foxp',\n",
       " 'evidence stress importance foxp expression treg function development maintenance protective immunity hiv htlv',\n",
       " 'based recent finding foxp may limit hiv htlv transcription interfering activation nf kappab creb pathway',\n",
       " 'however observing inhibitory effect absolute low level viral gene expression may persist cd cell particular regulatory cell known reservoir hiv htlv result accumulation viral protein either stimulate nf kappab creb activation directly inhibit foxp expression function',\n",
       " 'imbalance nf kappab creb activation caused viral gene product may crucial step pathogenesis virus induced immunological disorder aid ham tsp',\n",
       " 'future study directed identifying characterizing cellular protein interact foxp nucleus cytoplasm order better address foxp function guide development function regulatory cell health disease',\n",
       " 'cell culture',\n",
       " 'hek cell cultured dulbecco modified eagle medium invitrogen carlsbad california united state',\n",
       " 'jurkat cell primary human cd cell cultured rpmi medium invitrogen',\n",
       " 'medium supplemented mm l glutamine u ml penicillin mug ml streptomycin cambrex east rutherford new jersey united state fetal bovine serum atlanta biologicals norcross georgia united state',\n",
       " 'patient cell preparation',\n",
       " 'pbmcs prepared centrifugation ficoll hypaque gradient biowhittaker walkersville maryland united state eight ham tsp patient eight ac cell viably cryopreserved liquid nitrogen tested',\n",
       " 'ham tsp diagnosed according guideline',\n",
       " 'htlv seropositivity determined elisa abbott laboratory abbott park illinois united state confirmation western blot analysis genelabs technology redwood city california united state',\n",
       " 'blood sample obtained informed consent part clinical protocol reviewed approved nih institutional review panel',\n",
       " 'plasmid',\n",
       " 'expression vector encoding human foxp pcmv foxp ire egfp human foxp lacking forkhead fkh domain pcmv deltafkh ire egfp generous gift ziegler benaroya research institute pegfp c provided',\n",
       " 'lipinski niddk nih pcdna provided k',\n",
       " '',\n",
       " 'jeang niaid nih pcmv tax generous gift w',\n",
       " 'greene university california san francisco pgl luc pgl tkhrluc purchased promega madison wisconsin united state puc purchased stratagene la jolla california united state',\n",
       " 'hiv wt ltr hiv delta kappab ltr luciferase reporter vector constructed cloning xhoi hindiii ltr fragment phiv cat pdelta kappab hiv cat aid research reference reagent program niaid nih multiple cloning site pgl luc',\n",
       " 'nf kappab htlv ltr creb luciferase reporter pcmv p ha expression vector generously provided b',\n",
       " 'wigdahl drexel university college medicine',\n",
       " 'htlv pach infectious molecular clone described previously',\n",
       " 'pfr luc pfa cmv pfa creb pfa c jun purchased stratagene pfa tax encoding fusion protein consisting gal dna binding domain fused frame htlv tax constructed pcr amplification htlv tax using pcmv tax template bamhi bglii tagged primer',\n",
       " 'amplified insert digested ligated bamhi bglii site pfa cmv',\n",
       " 'plasmid content confirmed dna sequencing',\n",
       " 'isolation primary human cd cell',\n",
       " 'cd cell isolated cryopreserved healthy donor pbmcs negative selection cd cell isolation kit ii miltenyi biotech bergisch gladbach germany according manufacturer guideline',\n",
       " 'purity negatively selected cd cell consistently higher determined flow cytometry',\n",
       " 'transient expression luciferase assay',\n",
       " 'hek cell plated density x cell well six well culture plate bd bioscience san diego california united state prior transfection appropriate plasmid dna mug total using fugene transfection reagent roche basel switzerland',\n",
       " 'jurkat cell plated x cell well six well culture plate day transfection appropriate plasmid dna mug total using fugene transfection reagent',\n",
       " 'primary human cd cell x nucleofected specified plasmid dna mug total using human cell nucleofection kit amaxa gaithersburg maryland united state',\n",
       " 'forskolin mum calbiochem san diego california united state added experiment h posttransfection',\n",
       " 'cell harvested h posttransfection luciferase activity analyzed using dual luciferase reporter assay system promega monolight luminometer analytical luminescence laboratory san diego california united state according manufacturer guideline',\n",
       " 'pgl tkhrluc used internal control normalize transfection efficiency',\n",
       " 'nucleofected cd cell also monitored transfection efficiency cell viability h posttransfection follows',\n",
       " 'transfection efficiency routinely determined flow cytometric analysis egfp expression',\n",
       " 'cell viability determined staining amino actinomycin aad bd bioscience routinely',\n",
       " 'transfection efficiency cell viability nucleofected cd cell independent plasmid used',\n",
       " 'foxp htlv tax expression analysis real time rt pcr',\n",
       " 'real time rt pcr analysis foxp htlv tax expression performed previously described',\n",
       " 'briefly total rna extracted using rneasy mini kit qiagen valencia california united state according manufacturer guideline cdna synthesized reverse transcription using taqman gold rt pcr kit using random hexamer primer applied biosystems foster city california united state',\n",
       " 'foxp htlv tax mrna expression quantified real time pcr using abi prism sequence detection system applied biosystems',\n",
       " 'normalized value sample calculated relative quantity foxp htlv tax mrna expression divided relative quantity hprt mrna expression',\n",
       " 'value calculated following formula normalized foxp htlv tax expression ct value hprt ct value foxp htlv tax',\n",
       " 'real time pcr',\n",
       " 'real time pcr analysis htlv tax proviral load performed previously described',\n",
       " 'dna extracted x cell using puregene dna isolation kit gentra minneapolis minnesota united state ng sample dna solution analyzed system',\n",
       " 'htlv proviral dna load calculated following formula copy number htlv px per cell copy number px copy number beta actin x',\n",
       " 'western blot analysis',\n",
       " 'hek cell plated density x cell well six well culture plate bd bioscience prior transfection appropriate plasmid dna mug total using fugene transfection reagent roche',\n",
       " 'cell harvested h posttransfection whole cell lysates ripa buffer mm tris hcl ph mm nacl igepal np sodium deoxycholate mm edta mm dtt mm pmsf x complete mini protease inhibitor roche',\n",
       " 'protein concentration determined lowry assay bio rad hercules california united state colorimetric reaction read using versamax microplate reader molecular device sunnyvale california united state absorbance nm',\n",
       " 'size fractionation performed mug protein sample sd page protein transferred nitrocellulose pvdf membrane subjected immunoblotting using indicated antibody',\n",
       " 'foxp detected using rabbit anti human foxp polyclonal antibody ab ab abcam cambridge united kingdom anti rabbit igg hrp secondary antibody cell signaling technology beverly massachusetts united state',\n",
       " 'nf kappab p creb detected using rabbit anti human polyclonal p monoclonal antibody creb h cell signaling technology',\n",
       " 'beta actin detected using mouse monoclonal antibody ac sigma st louis missouri united state',\n",
       " 'coimmunoprecipitation analysis cell lysates precleared mul protein g plus agarose bead santa cruz biotechnology santa cruz california united state incubated mouse monoclonal anti ha antibody e cell signaling technology mul protein g plus agarose bead overnight',\n",
       " 'immunoprecipitates washed four time ripa buffer resuspended sd sample buffer heated degreesc min',\n",
       " 'protein treated described western blot analysis',\n",
       " 'transcription factor dna binding analysis tf elisa',\n",
       " 'creb atf dna binding activity analyzed transfactor profiling inflammation kit bd bioscience according manufacturer protocol',\n",
       " 'nuclear extract prepared hek cell transfected control vector egfp foxp expression vector ng presence absence forskolin mum h using transfactor extraction kit bd bioscience',\n",
       " 'protein concentration determined using biophotometer eppendorf hamburg germany',\n",
       " 'nuclear extract mug incubated preblocked well containing plate bound double stranded oligonucleotides corresponding atf creb consensus sequence emtgacatcaem',\n",
       " 'well washed incubated appropriate primary antibody washed incubated secondary antibody hrp labeled washed finally developed tmb substrate',\n",
       " 'colorimetric reaction read using versamax microplate reader molecular device absorbance nm',\n",
       " 'flow cytometric analysis foxp protein expression',\n",
       " 'cryopreserved pbmcs ham tsp patient htlv infected ac thawed washed facs buffer x pb nan fbs',\n",
       " 'cell x fixed sequential formaldehyde methanol fixation follows',\n",
       " 'cell carefully resuspended facs buffer fixed mul reagent fix perm kit caltag laboratory burlingame california united state room temperature min followed ml methanol min degreesc',\n",
       " 'cell washed twice permeabilized mul reagent b fix perm kit stained intracellular foxp mouse anti human foxp monoclonal antibody mug ab abcam appropriate isotype control min',\n",
       " 'cell washed twice stained cy conjugated goat anti mouse immunoglobulin f ab secondary antibody caltag laboratory additional min',\n",
       " 'cell washed twice stained surface cd expression pe labeled anti cd bd cd expression fitc labeled anti cd bd',\n",
       " 'cell washed twice analyzed facscalibur bd',\n",
       " 'data analysis performed using flowjo tree star ashland oregon united state',\n",
       " 'statistical analysis',\n",
       " 'mann whitney u test used compare data patient ham tsp ac',\n",
       " 'ebv latent membrane protein activates akt nfkappab stat b cell lymphoma',\n",
       " 'latent membrane protein lmp major oncoprotein epstein barr virus ebv',\n",
       " 'transgenic mouse lmp promotes increased lymphoma development mo age',\n",
       " 'study reveals lymphoma develops b lymphocyte population associated transformation older mouse',\n",
       " 'lymphoma cell deregulated cell cycle marker inhibitor akt nfkappab stat block enhanced viability lmp transgenic lymphocyte lymphoma cell vitro',\n",
       " 'lymphoma cell independent il stat signaling survival proliferation constitutively activated stat signaling',\n",
       " 'target also deregulated wild type b lymphoma arise spontaneously age predisposition',\n",
       " 'result suggest akt nfkappab stat pathway may serve effective target treatment ebv associated b cell lymphoma',\n",
       " 'author summary',\n",
       " 'epstein barr virus ebv linked development multiple cancer including post transplant lymphoma hodgkin disease nasopharyngeal carcinoma',\n",
       " 'latent membrane protein lmp expressed many ebv associated cancer responsible altered cellular growth property induced ebv infection',\n",
       " 'study reveals lmp induces lymphoma b lymphocyte cell type susceptible transformation aged mouse',\n",
       " 'lymphoma require akt nfkappab stat signaling enhanced growth survival',\n",
       " 'activation stat akt nfkappab signaling pathway likely underlies ability lmp promote malignant transformation',\n",
       " 'epstein barr virus ebv ubiquitous gamma herpesvirus infects human predominantly early age greater adult population infected ebv',\n",
       " 'ebv linked development b lymphocyte epithelial cell malignancy including burkitt lymphoma hodgkin disease hd nasopharyngeal carcinoma npc cancer linked immunosuppression including post transplant lymphoma aid associated lymphoma',\n",
       " 'vitro infection b lymphocyte ebv induces permanent growth transformation ability affect cell growth regulation likely contributes development cancer',\n",
       " 'many viral protein expressed transformed cell including ebv nuclear antigen latent membrane protein profound effect cell growth regulation required ebv latent infection b cell transformation',\n",
       " 'latent membrane protein lmp considered major oncoprotein ebv transforms rodent fibroblast tumorigenicity nude mouse expressed hd npc immunosuppression associated tumor',\n",
       " 'b lymphocyte lmp mimic cd signaling lmp cd essential ebv mediated b cell transformation',\n",
       " 'cd interacts cd ligand expressed activated cell induce b cell activation differentiation lmp act constitutive signal ligand independent oligomerization',\n",
       " 'lmp cd interact tumor necrosis factor receptor associated factor trafs leading activation nfkappab c jun n terminal kinase jnk p mapk signaling pathway',\n",
       " 'activation nfkappab required ebv induced b cell transformation inhibition rapidly result cell death',\n",
       " 'recent study indicate lmp also activates phosphatidylinositol kinase pi k akt signaling activation required lmp mediated transformation rodent fibroblast',\n",
       " 'vitro primary b cell maintained cd ligation combination il treatment',\n",
       " 'vivo cd signaling necessary germinal center gc formation mouse deficient cd cd l unable form gc response cell dependent antigen',\n",
       " 'membrane proximal distal cytoplasmic region cd bind traf trafs respectively necessary gc formation either region sufficient induce extrafollicular b cell differentiation restore low affinity antibody production',\n",
       " 'functionally lmp rescue cd deficient mouse restore immunoglobulin ig class switching likely lmp recruit similar traf molecule trafs traf c terminal activation region domain respectively',\n",
       " 'however lmp unable restore affinity maturation gc formation',\n",
       " 'several ebv transforming protein studied transgenic mouse model however lmp induces tumor development expressed control ig heavy chain promoter enhancer',\n",
       " 'lmp transgenic mouse iglmp express lmp b lymphocyte mouse older mo lymphoma develops increased incidence compared wild type control mouse suggesting lmp contributes tumor development',\n",
       " 'lmp lymphoma rearranged ig gene activated akt jnk p nfkappab specific activation nfkappab family member crel',\n",
       " 'study lmp transgenic lymphocyte lymphoma characterized growth property vitro determined',\n",
       " 'obtain pure population malignant lymphocyte enable detailed biochemical analysis example primary lymphoma inoculated passaged scid mouse',\n",
       " 'interestingly lymphoma development restricted b lymphocyte self replenishing population cell prone malignancy',\n",
       " 'lmp transgenic lymphocyte increased viability vitro viability increased addition il',\n",
       " 'contrast lmp positive negative lymphoma cell independent il co stimulation survival proliferation vitro complete absence activated stat il target',\n",
       " 'lymphoma also distinguished constitutive activation stat deregulation rb cell cycle pathway',\n",
       " 'inhibition pi k akt nfkappab stat signaling pathway blocked enhanced growth lmp transgenic malignant lymphocyte suggesting pathway required growth survival',\n",
       " 'appear target deregulated wild type b lymphoma arise spontaneously age predisposition',\n",
       " 'study reveals lmp promotes malignancy cell inherent ability proliferate akt nfkappab stat signaling pathway required growth stimulatory effect',\n",
       " 'high level lmp expression correlate development lymphoma',\n",
       " 'lmp expression iglmp mouse directed b cell control ig heavy chain promoter enhancer',\n",
       " 'previously shown transgenic mouse lmp expression restricted b b cell lymphoma detected greatly enlarged spleen',\n",
       " 'investigate whether lmp expression contributes lymphoma development b cell purified splenocytes positive selection using anti cd mac magnetic bead equivalent amount b cell analyzed immunoblotting',\n",
       " 'lmp detectable lmp transgenic b cell upon development lymphoma lmp expression stronger lymphoma analyzed concomitant appearance degradation product figure',\n",
       " 'determine whether higher level lmp detected due expansion malignant lymphocyte expression lmp spleen evaluated immunohistochemical staining',\n",
       " 'immunohistochemistry analysis spleen section detected lmp plasma membrane cell follicular white pulp circulating lymphocyte red pulp figure b',\n",
       " 'lmp expression heterogeneous strong lmp staining interspersed amongst background cell staining weakly lmp',\n",
       " 'upon development lymphoma lmp expression abundantly detected multiple focus intense lmp staining',\n",
       " 'demonstrates increased lmp detected immunoblotting upon malignant progression reflects increase lmp expression accumulation cell expressing high level lmp',\n",
       " 'correlation high lmp expression development lymphoma suggests progression lymphoma result increased level lmp',\n",
       " 'lmp promotes b lymphoma escape allelic exclusion',\n",
       " 'determine lmp signaling affect b cell differentiation immunophenotype lymphoma arise lmp expression surface ig expression heavy chain igm igg igd light chain kappa lambda analyzed flow cytometry',\n",
       " 'similar number naive igm igd igg splenic b cell strong bias towards kappa light chain detected wild type lmp transgenic mouse indicating lmp signaling affect b cell maturation unpublished data',\n",
       " 'flow cytometry analysis scid passaged wild type lmp transgenic lymphoma revealed igmhighigdlow phenotype figure indicative marginal zone b memory b cell',\n",
       " 'b cell separated cd b cd b b subset',\n",
       " 'differentiate cell type lymphoma cell analyzed b marker cd',\n",
       " 'tested lmp transgenic lymphoma displayed igmhighigdlow cd phenotype figure expression pattern distinguishes b cell',\n",
       " 'interestingly spontaneous wild type lymphoma also developed b cell figure',\n",
       " 'cell interesting population self replenishing increased likelihood become malignant aged mouse',\n",
       " 'analysis lmp transgenic mouse development lymphoma showed similar number splenic b cd cd b b b population cd cd indicating lmp affect b cell differentiation figure b',\n",
       " 'due allelic exclusion mature b cell exposed antigen typically express one heavy chain isotype igg ige iga either kappa lambda light chain',\n",
       " 'interestingly lmp transgenic lymphoma analyzed lymphoma doubly positive low level kappa lambda light chain figure',\n",
       " 'previous characterization lmp lymphoma revealed lymphoma clonal determined ig heavy chain rearrangement analysis kappa chain rearrangement figure sample analyzed study confirmed clonality',\n",
       " 'ass light chain expression passaged sample tested immunoblotting kappa lambda light chain figure c',\n",
       " 'interestingly low level expression light chain detected flow cytometry',\n",
       " 'low level expression may reflect limitation total number light chain expressed surface b cell',\n",
       " 'agreement flow cytometry analysis lmp transgenic lymphoma also positive kappa lambda light chain immunoblot analysis figure c confirming lymphoma express light chain',\n",
       " 'previous study mouse developed leukemia due expansion self reactive b cell determined b leukemia also doubly positive kappa lambda light chain',\n",
       " 'finding indicate expression lmp b cell promotes development malignancy result aberrant escape allelic exclusion',\n",
       " 'lmp promotes b cell survival proliferation vitro',\n",
       " 'primary b cell culture maintained cd ligation supplementation il',\n",
       " 'investigate whether lmp affect primary b cell survival proliferation splenocytes cultured presence absence il analyzed mt metabolic marker ethidium monoazide ema exclusion viability bromo deoxy uridine brdu incorporation proliferation',\n",
       " 'mt assay expected splenocytes wild type mouse survive even addition il due lack cd ligation figure',\n",
       " 'contrast lmp splenocytes increased metabolism even absence il enhanced upon addition il',\n",
       " 'wild type lmp transgenic lymphoma cell high level mt activity even absence il figure',\n",
       " 'lmp transgenic lymphoma cell approximately fold higher mt activity normal transgenic lymphocyte least fold higher control lymphoma',\n",
       " 'previously published lymphoma usually develops mouse mo age mouse sacrificed mo',\n",
       " 'age transgenic mouse without lymphoma ranged mo old',\n",
       " 'correlation age mt activity',\n",
       " 'ema exclusion cd gated b cell prepared two wild type two lmp transgenic mouse indicated fold increase viability lmp transgenic lymphocyte compared wild type lymphocyte',\n",
       " 'two example lmp transgenic lymphoma cell greatly increased viability increased il treatment indicating lymphoma cell independent il co stimulation figure b',\n",
       " 'enhancement mt activity observed lmp transgenic lymphoma cell addition il however ema exclusion reveal similar increase',\n",
       " 'could reflect difference il requirement metabolic activity versus viability lmp transgenic lymphoma cell',\n",
       " 'although expression lmp could enhance survival non malignant primary lymphocyte brdu incorporation revealed lmp expression alone sufficient induce proliferation culture unpublished data',\n",
       " 'lymphoma cell detectable level brdu incorporation detected flow cytometry figure c',\n",
       " 'interestingly lmp lymphoma cell significantly higher level proliferation comparison spontaneous lymphoma developed lmp negative littermate versus',\n",
       " 'higher level proliferation observed lymphoma express high table lmp l lmp l low table lmp l level lmp suggesting even small amount lmp sufficient induce dramatic effect proliferation',\n",
       " 'level proliferation enhanced upon il addition confirming il independence observed viability study figure c table',\n",
       " 'wild type lmp transgenic lymphoma cell require il stat signaling',\n",
       " 'investigate whether il independence due endogenous il expression il transcription assessed rnase protection assay rpa',\n",
       " 'il transcription detectable control rna faintly mouse lymphoma cell line k mu figure',\n",
       " 'however il transcription detectable cd mac purified b cell wild type lymphocyte unpublished data lmp transgenic lymphocyte lymphoma cell although gapdh l control effectively protected figure',\n",
       " 'activated stat pstat target il receptor pathway detected wild type lmp transgenic lymphocyte figure b',\n",
       " 'contrast pstat barely detected either wild type lmp transgenic lymphoma cell',\n",
       " 'however pathway disabled treatment lymphoma cell il induced stat phosphorylation figure c',\n",
       " 'although wild type lymphocyte maintained culture il supplementation alone figure slight enhancement mt activity could detected cell analyzed earlier time point figure versus figure post harvest',\n",
       " 'enhancement mt activity induced il wild type lymphocyte could neutralized addition il antibody figure',\n",
       " 'however neutralizing antibody il affect mt activity lmp transgenic lymphoma cell figure e',\n",
       " 'summary wild type lmp transgenic lymphoma cell grew independently il treatment require stat signaling',\n",
       " 'lmp upregulates il constitutively activates stat',\n",
       " 'identify cytokine may contribute increased survival growth lymphoma expression level panel cytokine screened cd mac purified b cell using rpa probe set il il il il il il il il ifngamma',\n",
       " 'expression level quantified phosphorimager normalized ribosomal housekeeping gene l',\n",
       " 'none tested cytokine detected wild type lymphocyte therefore cytokine l ratio set mouse b cell lymphoma line',\n",
       " 'transcription il il ifngamma reproducibly detected lmp transgenic lymphocyte lymphoma cell higher b cell lymphoma cell line k mu figure',\n",
       " 'significant difference expression il ifngamma lmp transgenic lymphocyte lymphoma cell suggesting upregulation il ifngamma induced lmp expression healthy lymphocyte unique property malignant lymphocyte',\n",
       " 'strikingly il b lymphocyte stimulatory cytokine increased fold wild type lmp transgenic lymphoma cell compared lmp transgenic lymphocyte figure',\n",
       " 'production il ifngamma associated induction cytotoxic effector response cell latently infected ebv',\n",
       " 'however transformation growth property induced ebv associated upregulation il hence effect il upregulation growth property lymphoma cell examined',\n",
       " 'immunoblot analysis indicated lmp transgenic lymphocyte wild type lmp transgenic lymphoma cell corresponding increased level phosphorylated alpha beta isoforms activated stat target il receptor figure b',\n",
       " 'however comparing lymphoma correlation level il induction level stat activation',\n",
       " 'suggests activation stat solely induced il stat activation may constitutive',\n",
       " 'additionally correlation level lmp expression level il induction figure',\n",
       " 'indicates induction il general property associated enhanced survival may indirectly affected lmp',\n",
       " 'neutralizing antibody il affect survival lymphoma cell determined mt assay unpublished data suggesting constitutive activation stat',\n",
       " 'confirmed immunoblot analysis presence anti il neutralizing antibody pstat level remained activated lymphoma cell isolated wild type lmp transgenic lymphoma figure c',\n",
       " 'exogenous addition il enhanced pstat activation constitutive level indicating lymphoma cell responsive il treatment figure c',\n",
       " 'mean although lymphoma cell constitutive stat activation may enhanced il induction',\n",
       " 'neutralizing effect anti il antibody confirmed pre incubation il anti il antibody compared rat igg isotype control figure c',\n",
       " 'nuclear translocation pstat consequence activation nuclear pstat detected immunohistochemistry staining spleen section control mouse',\n",
       " 'however nuclear pstat detectable lmp transgenic mouse wild type lymphoma detected homogeneously lmp transgenic lymphoma figure',\n",
       " 'constitutive activation pstat abundant nuclear stat suggests stat signaling contributes lmp mediated lymphoma development',\n",
       " 'lmp activates akt signaling deregulates rb cell cycle pathway',\n",
       " 'lmp transformation rodent fibroblast requires activation pi k akt',\n",
       " 'additionally activated pakt frequently detected npc neoplastic reed sternberg cell classical hd',\n",
       " 'determine akt signaling activated lmp transgenic mouse pakt several target assessed immunoblotting splenic cd mac purified b cell',\n",
       " 'lmp transgenic b cell increased level pakt compared wild type lymphocyte however progression lymphoma lmp positive negative lymphoma cell increase pakt level',\n",
       " 'akt target glycogen synthase kinase gsk inactivated phosphorylation however increased phosphorylated gsk detected transgenic lymphocyte almost absent lymphoma sample figure',\n",
       " 'finding indicates gsk target activated akt lmp transgenic lymphocyte lymphoma cell',\n",
       " 'similarly activation akt without phosphorylation gsk previously shown ebv positive hd',\n",
       " 'contrast wild type lymphocyte lacked activated akt detectable phosphorylated gsk',\n",
       " 'suggests additional pathway involved regulation gsk',\n",
       " 'identify potential akt target immunoblot analysis p mtor performed',\n",
       " 'activated p mtor increased lmp transgenic lymphocyte lymphoma cell indicating pathway affected lmp induced akt activation contribute lymphoma development figure b',\n",
       " 'akt also known phosphorylate induce degradation pro apoptotic forkhead family transcription factor leading cell cycle progression survival human tumor',\n",
       " 'immunoblot analysis splenic b cell consistently detect p foxo level signal target foxo degradation',\n",
       " 'hence degradation foxo assessed detection total foxo level',\n",
       " 'immunoblot analysis indicated total foxo level greatly decreased wild type lmp transgenic lymphoma figure b suggesting inhibition forkhead signaling pathway important target akt lymphoma development',\n",
       " 'however considering akt activation induce foxo degradation lmp transgenic b cell akt may sole regulator foxo may progression lymphoma requires modulation multiple pathway',\n",
       " 'forkhead family transcription factor known induce expression cdk inhibitor p',\n",
       " 'lmp transformed rodent fibroblast decreased expression p upregulation cdk subsequent phosphorylation inactivation tumor suppressor gene rb',\n",
       " 'investigate whether lmp affected cell cycle regulation rb pathway b cell immunoblot analysis prb cdk p performed splenic cd mac purified b cell',\n",
       " 'lmp transgenic b cell enhanced level prb concomitant stabilization total rb level cdk compared wild type b lymphocyte figure c',\n",
       " 'progression lymphoma wild type lmp transgenic lymphoma cell led increased level rb correspondingly high level cdk decreased level p figure c',\n",
       " 'data indicate rb pathway deregulated lmp transgenic lymphocyte lymphoma cell distinguished loss foxo decreased p',\n",
       " 'lmp promotes tumor growth survival activation akt nfkappab stat pathway',\n",
       " 'explore pathway required enhanced growth survival lmp induced lymphoma splenocytes wild type lmp transgenic mouse cultured presence inhibitor akt nfkappab stat mtor mapk assayed growth survival mt assay',\n",
       " 'previously shown wild type lymphocyte viable culture could tested inhibitor',\n",
       " 'however enhanced viability lmp transgenic lymphocyte effectively blocked treatment triciribine bay cucurbitacin slightly sb treatment rapamycin u ag figure',\n",
       " 'triciribine inhibits activation akt mum shown induce growth arrest cancer cell aberrant akt activity',\n",
       " 'effect triciribine cell growth transgenic lymphocyte lymphoma apparent low mum suggesting activation akt required survival growth lmp transgenic lymphocyte lymphoma cell figure',\n",
       " 'effect inhibitor assessed identifying phosphorylated akt stat total level ikappabalpha figure',\n",
       " 'treatment triciribine effectively blocked phosphorylation akt phosphorylated akt longer detected past mum',\n",
       " 'phosphorylated stat ikappabalpha still present mum',\n",
       " 'finding suggest triciribine specifically target akt akt activation required enhanced viability transgenic lymphocyte lymphoma cell',\n",
       " 'inhibition nfkappab signaling rapidly induces cell death ebv transformed lymphocyte',\n",
       " 'bay inhibitor nfkappab signaling also greatly decreased viability lmp transgenic lymphocyte lymphoma cell dos low mum mum cell completely nonviable figure',\n",
       " 'well within reported ic mum',\n",
       " 'phosphorylated akt stat total ikappabalpha still present treatment mum longer detected figure',\n",
       " 'finding suggests inhibition nfkappab induce cell death lmp transgenic lymphocyte lymphoma cell without significant effect activation akt stat',\n",
       " 'cucurbitacin inhibits activation stat suppressing activation kinase jak',\n",
       " 'shown selectively inhibit growth tumor constitutively activated stat',\n",
       " 'similarly lmp transgenic lymphocyte lymphoma cell susceptible cucurbitacin treatment starting mum dose corresponds closely reported ic nm figure',\n",
       " 'phosphorylated akt stat total ikappabalpha detectable past mum higher dos protein level greatly decreased indicative total loss viability figure',\n",
       " 'second reported inhibitor stat ag effect growth figure activation akt stat level ikappabalpha also affected figure',\n",
       " 'finding suggest inhibition stat induce cell death lmp transgenic lymphocyte lymphoma cell stat inhibition also considerable crossover effect akt nfkappab signaling',\n",
       " 'lmp also shown activate jnk p mapk pathway lmp transgenic lymphocyte mildly susceptible growth inhibition sb inhibitor p mapk u inhibitor mek activity',\n",
       " 'however effect sb apparent high dos mum much higher reported ic mum suggesting p mapk significantly contribute enhanced viability lmp transgenic lymphocyte lymphoma cell figure',\n",
       " 'interestingly wild type lmp transgenic lymphoma similarly susceptible triciribine bay cucurbitacin treatment sb ag u treatment suggesting activation akt nfkappab stat mapk pathway characteristic associated malignant transformation figure rapamycin inhibitor mtor affect viability transgenic lymphocyte lymphoma cell confirming mtor targeted akt activation lmp transgenic lymphocyte malignant lymphoma cell figure',\n",
       " 'study defines oncogenic property lmp promoting b cell lymphomagenesis',\n",
       " 'lmp transgenic mouse higher incidence lymphoma progression lymphoma correlate higher expression level lmp figure b suggesting lmp directly involved tumor development',\n",
       " 'table summarizes biological molecular property identified wild type lmp transgenic lymphoma',\n",
       " 'although many molecular property studied similar wild type lmp transgenic lymphoma distinguishing biological property namely ability lmp transgenic lymphoma induce higher level survival proliferation',\n",
       " 'interestingly although lmp transgenic mouse develop lymphoma b cell type spontaneous wild type lymphoma figure signaling effect induced lmp may explain enhanced promotion lymphomagenesis',\n",
       " 'since cd deficient mouse decreased number igmhighigdlow cell phenotype associated b marginal zone memory b cell mimicry cd signaling lmp could possibly contribute expansion b cell',\n",
       " 'noteworthy expression lmp transgenic mouse shown inhibit formation gc preventing typical b cell antigen driven selection expansion',\n",
       " 'lack gc reaction may contribute bias lmp transgenic mouse towards b cell lymphoma',\n",
       " 'interestingly lmp signaling also favor development b cell occurs absence transformation',\n",
       " 'result suggest mimicry b cell receptor signaling lmp promotes b cell differentiation transformation',\n",
       " 'promotion b differentiation may account ability lmp exacerbate autoimmunity bypass anergy induction',\n",
       " 'contrast preponderance tumor b origin reflect effect lmp signaling b cell differentiation splenic b cell healthy lmp transgenic mouse contain similar number b b cell wild type mouse',\n",
       " 'support lack effect differentiation b versus b cell thought independent cd signaling',\n",
       " 'b cell constitute predominant lymphocyte population peritoneal pleuropericardial cavity b cell mainly found spleen lymph node peripheral blood',\n",
       " 'b cell produce main source igm iga antibody serum involved cell independent response common microbial antigen',\n",
       " 'importantly b cell unique capacity self replenish also predisposed transformation',\n",
       " 'clonal expansion b cell detected aging mouse mo age b cell thought murine progenitor b cell chronic lymphocytic leukemia',\n",
       " 'data presented study indicate although lmp expressed b lymphocyte transgenic mouse malignancy develops specific subset b cell',\n",
       " 'elevated expression lmp b cell activation specific pathway apparently induce malignant growth',\n",
       " 'pathway also become sporadically activated aged mouse also result lymphoma development',\n",
       " 'similar ebv associated cancer vivo pathway activated lmp also activated le prevalent ebv negative form cancer',\n",
       " 'thus contribution ebv lmp tumor development apparently continuous activation pathway also sporadically activated contribute tumor development',\n",
       " 'lymphoma marked upregulation il constitutive activation stat signaling requirement activation akt nfkappab stat pathway figure',\n",
       " 'induction il associated transformation b lymphoma mouse frequently associated ebv positive b cell malignancy acting b cell growth factor',\n",
       " 'addition lmp shown stimulate il expression burkitt lymphoma cell line',\n",
       " 'suggests although stat constitutively activated lymphoma cell induction il may enhance stat activation may contribute il responsive signaling pathway',\n",
       " 'lmp activates canonical non canonical pathway nfkappab signaling inhibition nfkappab blocked survival lmp transgenic lymphocyte lmp positive negative lymphoma cell',\n",
       " 'nfkappab pi k signaling crucial cd induced proliferation mouse deficient crel p regulatory subunit pi k unresponsive mitogenic stimulus including cd ligation',\n",
       " 'previously shown crel specifically activated wild type lmp transgenic lymphoma suggesting activation crel associated b cell transformation',\n",
       " 'observation suggest similar cd induced proliferation lmp induces proliferation pi k mediated activation akt activation nfkappab component crel',\n",
       " 'cd also induces downregulation cell cycle inhibitor p pi k dependent manner lmp lymphoma cell also decreased level p phosphorylation rb increased cdk figure c',\n",
       " 'although lmp shown deregulate rb pathway epithelial cell knowledge first demonstration property b lymphocyte',\n",
       " 'requirement akt activation confirmed striking inhibition lymphoma viability triciribine akt inhibitor',\n",
       " 'however activated pakt lead phosphorylation inactivation downstream target gsk figure',\n",
       " 'effect also described ebv positive hd biopsy',\n",
       " 'contrast rapamycin u sb affect survival lmp transgenic lymphocyte wild type lmp transgenic lymphoma cell figure c e',\n",
       " 'lack effect rapamycin confirmed absence activated p mtor level figure b',\n",
       " 'finding suggest akt target contribute malignant progression',\n",
       " 'one key target likely inhibition fox transcription factor',\n",
       " 'repression pro apoptotic transcription factor foxo pi k dependent manner inhibit expression bcl transcription factor necessary gc formation',\n",
       " 'shown overstimulation cd signaling agonistic antibody inhibits gc formation',\n",
       " 'similarly due mimicry cd signaling transgenic lmp mouse also defective gc formation',\n",
       " 'constitutive signaling lmp likely block gc formation downregulation bcl',\n",
       " 'interestingly clinical study indicate expression lmp bcl mutually exclusive non hd classical hd',\n",
       " 'thus lmp transgenic lymphoma mirror aspect ebv induced hd',\n",
       " 'although activation akt lack fox lymphoma cell suggest lmp affect bcl gc formation pathway regulation forkhead target involved cell cycle progression p cyclind likely contribute malignant transformation',\n",
       " 'indeed loss foxo expression lymphoma cell correlated loss p figure b c',\n",
       " 'cyclind also shown upregulated lmp release foxo mediated repression',\n",
       " 'summary transgenic model lymphomagenesis lmp promotes malignancy b cell population predisposed clonal expansion age',\n",
       " 'malignant lymphocyte distinguished constitutively active stat signaling decreased p activated akt nfkappab pathway property associated promoting growth survival b lymphocyte',\n",
       " 'importantly akt nfkappab stat pathway critically required growth survival malignant lymphocyte well healthy lmp transgenic lymphocyte',\n",
       " 'growth ebv transformed lymphocyte also requires activation nfkappab study provide insight lmp contributes ebv associated transformation',\n",
       " 'transgenic lymphoma mirror multiple aspect ebv induced tumor suggest vivo property lmp major factor development cancer',\n",
       " 'transgenic mouse',\n",
       " 'generation lmp mouse ig heavy chain promoter enhancer described previously maintained heterozygote balbc background',\n",
       " 'lmp mouse genotyped southern blot pcr analysis tail dna described previously',\n",
       " 'spleen liver section fixed paraformaldehyde embedded paraffin mum section stained hematoxylin eosin histopathological analysis',\n",
       " 'lymphoma passaged intraperitoneal injection x splenocytes scid mouse sacrificed upon development extended abdomen',\n",
       " 'animal housed association assessment accreditation animal care approved animal facility university north carolina chapel hill',\n",
       " 'protocol approved institutional animal care use committee',\n",
       " 'isolation growth b cell',\n",
       " 'splenocytes prepared homogenizing spleen tissue two frosted slide debris filtered mum cell strainer',\n",
       " 'erythrocyte lysed using ammonium chloride solution stemcell technology min ice washed twice pb',\n",
       " 'b cell isolated using cd mac bead according manufacturer instruction miltenyi biotec grown iscove medium supplemented heat inactivated fetal bovine serum antibiotic antimycotic gibco',\n",
       " 'splenocytes isolated scid passaged lymphoma consisted b cell determined flow cytometry hence purified cd mac bead',\n",
       " 'splenocytes seeded x cell ml applicable recombinant mouse il added ng ml recombinant mouse il ng ml rat igg anti mouse il rat igg isotype control mug ml r system',\n",
       " 'brdu incorporation assay splenocytes pulsed h mum brdu post harvest',\n",
       " 'mt assay',\n",
       " 'dimethylthiazol yl carboxymethoxyphenyl sulfophenyl h tetrazolium inner salt mt cell cytotoxicity proliferation assay performed using celltiter aqueous one solution cell proliferation assay promega according manufacturer instruction',\n",
       " 'il study splenocytes cultured presence absence ng ml il',\n",
       " 'cell seeded day triplicate well plate x cell ml mul per well',\n",
       " 'mt reagent added h absorbance read nm value plotted subtracted blank',\n",
       " 'neutralization assay splenocytes seeded x cell ml mul per well day harvest il rat igg anti mouse il rat igg isotype control r system added concentration indicated figure',\n",
       " 'culture pulsed h mt reagent post seeding',\n",
       " 'inhibitor study splenocytes seeded x cell ml mul per well day harvest inhibitor added concentration indicated figure',\n",
       " 'inhibitor bay rapamycin triciribine u sb cucurbitacin purchased emd bioscience',\n",
       " 'non malignant splenocyte culture b cell activation induced mug ml goat f ab anti mouse igm jackson immunoresearch',\n",
       " 'culture pulsed h mt reagent post seeding',\n",
       " 'immunohistochemistry',\n",
       " 'paraffin embedded spleen section deparaffinized histoclear national diagnostics rehydrated graded ethanol',\n",
       " 'section antigen retrieved microwaving citrate buffer ph min lmp staining min pstat staining',\n",
       " 'lmp staining section blocked bsa cold fish skin gelatin followed streptavidin biotin block vector lab',\n",
       " 'rat igg anti lmp clone g g ascenion used dilution tissue culture supernatant followed mug ml biotinylated mouse f ab anti rat igg h l pre adsorbed mouse serum jackson immunoresearch mug ml streptavidin alkaline phosphatase conjugate jackson immunoresearch',\n",
       " 'stain developed bcip nbt counterstained nuclear fast red dako',\n",
       " 'phospho stat detected mug ml pstat antibody tyr cell signaling detected anti rabbit poly hrp ihc detection kit chemicon',\n",
       " 'flow cytometry',\n",
       " 'one million splenocytes stained appropriate primary antibody unconjugated conjugated fitc pe apc diluted stain buffer pb fbs',\n",
       " 'brdu detection fitc brdu flow kit used instructed manufacturer bd bioscience',\n",
       " 'briefly cell exposed ema molecular probe exclusion dead cell stained surface antigen fixed paraformaldehyde permeabilized saponin',\n",
       " 'expose brdu epitope cell treated dnase stained fitc conjugated anti brdu antibody',\n",
       " 'flow cytometry performed facscalibur using cellquest program becton dickinson',\n",
       " 'analysis conducted summit v program dako',\n",
       " 'rnase protection assay',\n",
       " 'total rna isolated cd mac purified b cell using rneasy midi purification kit dnase treatment according manufacturer instruction qiagen',\n",
       " 'mck probe template set bd bioscience labeled using vitro transcription kit according manufacturer instruction bd bioscience',\n",
       " 'briefly ng mck probe set labeled alpha p utp using rna polymerase purified using sephadex g column nucaway spin column ambion',\n",
       " 'labeled probe quantitated using scintillation counter x cherenkov cpm used hybridize mug total rna using rpa kit bd bioscience',\n",
       " 'sample denatured degreesc hybridized degreesc overnight',\n",
       " 'single stranded rna digested mixture rnase precipitated using isopropanol',\n",
       " 'protected probe resolved denaturing acrylamide gel dried imaged using phosphorimager molecular dynamic',\n",
       " 'densitometry performed using imagequant tl v software ge healthcare',\n",
       " 'pcr analysis kappa chain rearrangement',\n",
       " 'dna isolated splenocytes using dneasy tissue kit qiagen rnase treatment',\n",
       " 'pcr reaction contained ng genomic dna mum primer mm dntps u taq dna polymerase neb performed x thermopol buffer neb',\n",
       " 'primer used vkappacon jkappa degree described previously',\n",
       " 'pcr condition degreesc min cycle degreesc degreesc degreesc min followed cycle degreesc min',\n",
       " 'immunoblot analysis',\n",
       " 'whole cell lysates prepared radioimmunoprecipitation assay ripa buffer mm tris hcl ph mm nacl mm edta np sd sodium deoxycholate supplemented mm phenylmethylsulfonyl fluoride mm na vo protease phosphatase inhibitor cocktail sigma',\n",
       " 'crude lysates centrifuged rpm min degreesc supernatant collected analysis',\n",
       " 'protein concentration determined bio rad dc protein assay system',\n",
       " 'lysates boiled presence beta mecaptoethanol separated denaturing sd page transferred mum optitran membrane schleicher schuell bio rad transfer unit',\n",
       " 'membrane immunoblotted appropriate primary antibody followed horseradish peroxidase tagged secondary antibody amersham bioscience dako detected supersignal west pico system pierce',\n",
       " 'antibody',\n",
       " 'fitc conjugated goat anti mouse igm rat igg akappa anti mouse igd clone rat igg kappa anti mouse kappa clone rat igg bkappa anti mouse lambda clone jc pe conjugated goat anti mouse igg un conjugated goat anti mouse kappa anti mouse lambda purchased southern biotech',\n",
       " 'apc conjugated rat igg akappa anti mouse cd clone pe conjugated mouse igg akappa anti lmp clone un conjugated mouse igg anti rb clone anti cdk clone purchased bd bioscience',\n",
       " 'pe conjugated rat igg akappa anti mouse cd clone purchased ebioscience',\n",
       " 'rat anti lmp clone g g e g purchased ascenion',\n",
       " 'rabbit anti pakt ser anti pgsk alpha beta ser anti pstat tyr anti pstat tyr anti pmtor ser anti stat anti akt anti foxo anti p purchased cell signaling',\n",
       " 'rabbit anti stat h anti ikappabalpha c goat anti beta actin purchased santa cruz biotechnology',\n",
       " 'mouse igg kappa anti gsk purchased upstate biotechnology',\n",
       " 'rabbit anti prb thr purchased emd bioscience',\n",
       " 'accession number',\n",
       " 'genbank http www ncbi nlm nih gov genbank accession number ebv genome sequence aj',\n",
       " 'gene identifier lmp bnlf',\n",
       " 'high lmp expression correlate development lymphoma',\n",
       " 'lmp expression shown immunoblotting purified b cell cd b immunohistochemistry staining spleen tissue wild type wt lmp transgenic mouse',\n",
       " 'lymphoma identified number',\n",
       " 'arrow indicate lmp specific band degradation product well non specific band',\n",
       " 'actin used loading control',\n",
       " 'b white red pulp shown architecture lost upon development lymphoma',\n",
       " 'scale bar mum',\n",
       " 'lmp promotes b lymphoma escape allelic exclusion',\n",
       " 'flow cytometry analysis splenocytes wt lmp transgenic lymphoma pan b cell cd b cell cd ig heavy chain igm igd light chain kappa lambda marker',\n",
       " 'shown result wt lymphoma lmp transgenic lymphoma',\n",
       " 'analysis repeated four lmp transgenic lymphoma showing similar b phenotype lymphoma also doubly positive kappa lambda light chain',\n",
       " 'b flow cytometry analysis wt lmp transgenic splenocytes b cd cd b b b subset cd cd',\n",
       " 'percentage b b b b subset shown quadrant',\n",
       " 'analysis repeated three wt two lmp transgenic mouse similar result',\n",
       " 'c immunoblot analysis kappa lambda light chain b cell cd purified wt lmp transgenic mouse',\n",
       " 'actin used loading control',\n",
       " 'lmp promotes b cell survival proliferation vitro',\n",
       " 'mt assay splenocytes wt lmp transgenic mouse',\n",
       " 'splenocytes cultured presence grey bar absence black bar il',\n",
       " 'result mean sem triplicate sample averaged multiple mouse n number mouse analyzed follows n wt lymphocyte wt lymphoma n lmp transgenic lymphocyte n lmp transgenic lymphoma',\n",
       " 'b ema exclusion cd gated splenocytes wt lmp transgenic mouse showing percentage viable b cell cultured white bar without black bar il',\n",
       " 'c flow cytometry analysis incorporated brdu wt lmp transgenic lymphoma cell cultured without il',\n",
       " 'shown result wt lymphoma lmp transgenic lymphoma',\n",
       " 'percentage cell quadrant shown',\n",
       " 'summary analysis performed wild type lmp transgenic lymphoma',\n",
       " 'wild type lmp transgenic lymphoma cell survive independently il stat signaling culture',\n",
       " 'rnase protection assay il mrna purified b cell cd wt lmp transgenic splenocytes',\n",
       " 'l gapdh housekeeping gene used loading control',\n",
       " 'arrow indicates position protected probe',\n",
       " 'b c immunoblot analysis wt lmp transgenic mouse activated pstat b purified b cell cd time harvest c whole splenocytes cultured without il',\n",
       " 'b actin used loading control white line indicates intervening lane spliced',\n",
       " 'e mt assay wt lymphocyte e lmp transgenic lymphoma cell cultured il neutralizing antibody il rat igg isotype control indicated concentration',\n",
       " 'shown result lmp transgenic lymphoma',\n",
       " 'result mean sem triplicate sample single representative experiment repeated twice similar result',\n",
       " 'lmp upregulates il expression constitutively activates stat',\n",
       " 'relative expression il il ifngamma mrna wt lmp transgenic b cell cd detected rnase protection assay',\n",
       " 'mouse lymphoma cell line k mu used control',\n",
       " 'expression level quantified phosphorimager value normalized ribosomal housekeeping gene l',\n",
       " 'cytokine l ratio set mouse b cell lymphoma line',\n",
       " 'b c immunoblot analysis activated pstat purified b cell cd wt lmp transgenic mouse b time harvest c h culture without il neutralizing antibody il rat igg isotype control',\n",
       " 'c shown result wt lymphoma lmp transgenic lymphoma',\n",
       " 'arrow indicate position alpha beta isoforms stat',\n",
       " 'actin used loading control',\n",
       " 'immunohistochemistry detection activated nuclear pstat spleen wt lmp transgenic mouse',\n",
       " 'scale bar mum',\n",
       " 'lmp activates akt signaling deregulates rb cell cycle pathway',\n",
       " 'b immunoblot analysis purified b cell cd spleen wt lmp transgenic mouse akt signaling probing activated pakt downstream target including inactivated pgsk alpha beta b activated p mtor total level foxo',\n",
       " 'arrow indicate position alpha beta isoforms gsk',\n",
       " 'white line indicates intervening lane spliced',\n",
       " 'c immunoblot analysis cell cycle protein regulating rb pathway probing activated prb total level cdk cdk inhibitor p',\n",
       " 'actin used loading control',\n",
       " 'akt nfkappab stat signaling required growth survival lymphoma cell',\n",
       " 'mt assay splenocytes b wt c lmp transgenic lymphoma e f lmp transgenic lymphoctyes',\n",
       " 'splenocytes cultured without inhibitor nfkappab bay mtor rapamycin akt triciribine mek u p sb stat cucurbitacin ag indicated concentration',\n",
       " 'result mean sem triplicate sample',\n",
       " 'shown result b wt lymphoma c lmp transgenic lymphoma e f one two lmp transgenic mouse analyzed',\n",
       " 'analysis repeated lmp transgenic lymphoma yielding similar result',\n",
       " 'analysis akt nfkappab stat pathway contribution growth survival lymphoma cell',\n",
       " 'immunoblot analysis wild type lmp transgenic lymphoma akt nfkappab stat signaling treatment akt inhibitor triciribine b nfkappab inhibitor bay stat inhibitor c cucurbitacin ag indicated concentration',\n",
       " 'arrow indicate position alpha beta isoforms stat',\n",
       " 'actin used loading control',\n",
       " 'lymphoma clonal kappa light chain rearrangement',\n",
       " 'pcr analysis total spleen genomic dna mouse kappa locus',\n",
       " 'primer designed amplify v j rearrangement including vj vj vj vj',\n",
       " 'specific pcr product four rearrangement corresponding calculated size indicated arrow',\n",
       " 'band size due non specific product',\n",
       " 'tumor present background normal spleen cell single band appears brighter specific band',\n",
       " 'wild type lmp transgenic lymphocyte positive four v j rearrangement serve positive control',\n",
       " 'clonality determined appearance predominant band correct size single v j rearrangement',\n",
       " 'kb pdf',\n",
       " 'click additional data file',\n",
       " 'runx box protein cooperate establish transcriptional program effector ctls',\n",
       " 'activation naive cd cell antigen induces differentiation effector cytolytic lymphocyte ctls',\n",
       " 'ctls lyse infected aberrant target cell exocytosis lytic granule containing pore forming protein perforin family protease termed granzymes',\n",
       " 'show effector ctl differentiation occurs two sequential phase vitro characterized early induction bet late induction eomesodermin eomes box transcription factor regulate early late phase interferon ifn gamma expression respectively',\n",
       " 'addition demonstrate critical role transcription factor runx ctl differentiation',\n",
       " 'runx regulates eomes expression well expression three cardinal marker effector ctl program ifn gamma perforin granzyme b',\n",
       " 'data point existence elaborate transcriptional network runx initially induces cooperates box transcription factor regulate gene transcription differentiating ctls',\n",
       " 'naive cd cell differentiate effector ctls ability lyse antigen bearing target cell exocytosis lytic granule containing perforin granzymes produce inflammatory cytokine ifn gamma tnf upon restimulation tcr',\n",
       " 'vivo experiment elucidated many critical parameter governing development evolution primary ctl response',\n",
       " 'study used vitro system developed study cd cell differentiation define molecular basis effector ctl differentiation',\n",
       " 'box transcription factor eomesodermin eomes bet needed important aspect effector memory ctl differentiation',\n",
       " 'uninfected mouse compound deletion tbx encoding bet eomesodermin gene associated selective loss cd cell il rbeta high memory phenotype',\n",
       " 'mouse deficient bet eomes cell impaired expression cytolytic mediator manifest poor cytolytic activity fail control acute lymphocytic choriomeningitis virus infection',\n",
       " 'nevertheless specific role bet eomes clonal expansion ctl differentiation yet resolved particular known whether transcription factor function redundantly control effector cd cell differentiation whether directly targeting specific effector cytokine cytolytic gene',\n",
       " 'runx protein family three dna binding transcription factor control thymocyte differentiation cd cd lineage decision',\n",
       " 'runx perforin mrna expressed double positive dp thymocytes cd single positive sp thymocytes cd sp cell',\n",
       " 'although runx expressed naive cd cell expression regulated th cell differentiation runx influence th cell differentiation function direct regulation il ifng cytokine gene',\n",
       " 'contrast three runx protein expressed mature cd cell runx deficient cd cell show reduced cytolytic activity',\n",
       " 'therefore tested whether runx influenced cytolytic cell differentiation',\n",
       " 'report show runx box factor synergistically regulate ctl differentiation function',\n",
       " 'bet induced quickly upon tcr stimulation required early programming cytokine production whereas eomes induced later differentiation sustains ifn gamma expression',\n",
       " 'runx required eomes perforin expression eomes runx bind prf locus contrast perforin expression unaffected bet deficient cell',\n",
       " 'cell lacking runx show decreased expression ifn gamma granzyme b runx also bind promoter region ifng gzmb gene',\n",
       " 'collectively result provide evidence complex transcriptional network runx primary regulator gzmb expression synergizes bet eomes respectively promote transcription ifng prf gene',\n",
       " 'cell culture system monitor effector ctl differentiation',\n",
       " 'used simple cell culture system examine kinetics effector gene expression cd cell differentiation',\n",
       " 'naive cd cell p tcr transgenic mouse activated anti cd anti cd splenic apc presence gp peptide cultured medium containing u ml recombinant human il rhil',\n",
       " 'used tcr transgenic mouse experiment provide reliable source cd cell truly naive however chose stimulate cell antigen experiment avoid contamination protein nucleic acid derived apc',\n",
       " 'minor difference gene expression differentiation induced antigen apc versus anti cd anti cd major conclusion presented report activating condition',\n",
       " 'culture condition activated cd cell expanded exponentially accumulated',\n",
       " 'limited analysis first activation period coincides clonal expansion cd cell activation vivo',\n",
       " 'distinct expression kinetics perforin granzyme b ctl development culture',\n",
       " 'experiment revealed clear difference kinetics perforin granzyme b cytokine expression cd cell activation fig',\n",
       " 'naive cell showed detectable expression perforin mrna well perforin protein fig',\n",
       " 'relative expression naive cell perforin prf mrna expression increase appreciably day showed reproducible decrease day followed robust reexpression day fig',\n",
       " 'contrast granzyme b gzmb mrna low undetectable naive cell strongly regulated day stimulation increased progressively day fig b similarly granzyme b protein expressed day remained high day fig e',\n",
       " 'expected small fraction naive cell expressed cytokine ifn gamma tnf response stimulation capacity increased significantly differentiated cell fig e see also fig',\n",
       " 'evaluated antigen dependent cytolytic function short term assay target cell death measured within h fig f',\n",
       " 'limiting duration tcr stimulation strategy minimizes cytolysis secondary new gene expression period assay',\n",
       " 'naive cell display significant cytolytic function short term assay unpublished data likely express immature unprocessed form perforin lack capacity degranulate',\n",
       " 'even activation cell showed poor cytolytic activity fig f striking contrast capacity efficient cytokine production fig e',\n",
       " 'cell cultured day displayed robust cytotoxicity judged ability induce apoptosis large number target cell fig f',\n",
       " 'result show strong priming stimulus tcrs co stimulatory receptor vitro granzyme b expression ability produce effector cytokine programmed early whereas perforin expression cytolytic function induced later phase clonal expansion il',\n",
       " 'therefore two major effector function ctl cytokine production cytolytic activity intrinsically coregulated',\n",
       " 'distinct kinetics bet eomes expression ctl differentiation',\n",
       " 'box transcription factor bet eomes linked regulation gene encoding effector cytokine e g ifng gene important cytolytic function e g prf gzmb',\n",
       " 'investigated kinetics expression transcription factor vitro culture fig',\n",
       " 'bet mrna protein detectable naive cd cell strongly induced upon tcr priming day remained expressed day differentiation fig c quantified fig b',\n",
       " 'contrast eomes expression low undetectable mrna protein level naive cd cell tcr priming culture modest effect expression day fig c',\n",
       " 'strong induction eomes mrna protein observed day later fig c',\n",
       " 'bet mrna expression slightly preceded expression gzmb mrna similarly eomes mrna protein expressed approximately ahead reexpression perforin mrna protein respectively fig b',\n",
       " 'detailed kinetic analysis suggested culture condition bet eomes contribute distinct aspect gene transcription ctl differentiation',\n",
       " 'bet required early ifn gamma production data suggested eomes might function early period rather might contribute later control perforin expression',\n",
       " 'data seemed consistent model tcr signal induce bet turn induces ifn gamma possibly granzyme b subsequently eomes induced period clonal expansion il activates perforin expression',\n",
       " 'perforin granzyme b expression appreciably regulated bet',\n",
       " 'test model outlined previous paragraph directly compared expression ifn gamma perforin granzyme b cd cell wt tbx bet deficient mouse',\n",
       " 'expected naive tbx cd cell produced ifn gamma poorly upon activation fig',\n",
       " 'notably deleterious effect bet deficiency observed differentiating cd cell day culture almost completely mitigated day fig',\n",
       " 'likely reflected compensation eomes strongly induced day fig',\n",
       " 'contrast bet deficient cell cultured showed defect perforin mrna expression fig b compare lane',\n",
       " 'consistently observed modest reduction gzmb mrna bet deficient cell fig b compare lane translate decrease expression granzyme b protein fig c',\n",
       " 'examine role eomes transduced naive cd cell wt tbx mouse retrovirus containing internal ribosome entry site ire gfp either empty encoded strongly transactivating version eomes eo vp expanded culture condition',\n",
       " 'eo vp empty gfp retrovirus increased perforin expression wt bet deficient cd cell fig b lane',\n",
       " 'expected eo vp also rescued early defect ifn gamma production observed bet deficient cd cell fig',\n",
       " 'however eo vp induce gzmb mrna expression either wt bet deficient cell thus partial bet dependence gzmb mrna expression compensated eo vp',\n",
       " 'runx control multiple aspect ctl differentiation program part induction eomes',\n",
       " 'runx highly expressed peripheral cd cell bet runx cooperation observed earlier cd cell examined role runx effector ctl differentiation',\n",
       " 'isolated cd cell runx ko mouse outbred icr background wt runx littermates positive selection anti cd magnetic bead fig available http www jem org cgi content full jem dc',\n",
       " 'strikingly runx cd cell strongly impaired ability differentiate effector ctls judged expression perforin granzyme b ifn gamma fig',\n",
       " 'compared wt cell perforin mrna protein expression essentially undetectable runx cell day culture fig b',\n",
       " 'runx cell also detectable eomes expression contrast bet expression unimpaired fig',\n",
       " 'furthermore runx required maximal production ifn gamma tnf il cd cell restimulated day fig c',\n",
       " 'previously reported th cell differentiation regulated feed forward loop bet regulated early induces runx bet runx cooperate induce ifn gamma silence il thus promoting stable differentiation toward th lineage',\n",
       " 'runx appeared necessary eomes induction fig b kinetics eomes expression paralleled perforin expression fig c overexpression eo vp either wt bet deficient cell led increase perforin ifn gamma expression fig b asked whether ctl differentiation also potentially regulated feed forward loop involving two class runx box transcription factor',\n",
       " 'specifically asked whether runx necessary eomes induction cooperated eomes regulate transcription effector ctl marker perforin ifn gamma granzyme b',\n",
       " 'test hypothesis used chromatin immunoprecipitation chip assay ask whether eomes runx bound regulatory region prf ifng gzmb gene fig',\n",
       " 'protein associated gene regulatory region differentiated ctls',\n",
       " 'runx bound prf gzmb transcription start site tss known il responsive enhancer located near kb prf gene distal ctl specific dnase hypersensitive site prf locus ifng promoter near tss previously reported th cell several dnase hypersensitive site ifng locus fig depicted',\n",
       " 'eomes bound primarily prf tss kb enhancer binding substantially greater observed promoter il rb gene known direct target eomes comparable observed ifng tss known target box protein th cd cell fig',\n",
       " 'determine whether runx controlled expression ctl effector gene induction eomes retrovirally expressed runx eo vp cd cell runx mouse',\n",
       " 'limited number cd cell mouse saw difference runx cd cd sp cd cd dp cell previous experiment used total runx cd cell without fractionation recipient retroviral transduction',\n",
       " 'reconstitution runx cd cell runx restored expression eomes well perforin granzyme b ifn gamma fig b',\n",
       " 'addition runx cell showed compensatory regulation runx suppressed upon reconstitution runx indicating runx target repression runx',\n",
       " 'notably eo vp regulate perforin expression expressed runx cell even though restored capacity induce ifn gamma expression upon tcr restimulation fig b',\n",
       " 'result suggests strongly perforin expression requires runx eomes',\n",
       " 'expected defect perforin granzyme b expression runx cd cell showed defective cytolytic activity mixed lymphocyte reaction',\n",
       " 'however tcr stimulated runx cd cell effective wt cell killing tumor cell redirected ctl assay',\n",
       " 'furthermore cd cell peritoneal cavity runx mouse immunized certain tumor cell effectively killed target',\n",
       " 'therefore although activation perforin granzyme b machinery defective runx cd cell cell entirely devoid cytolytic activity could still effectively kill target possibly alternative mechanism fa fa ligand pathway',\n",
       " 'runx box factor control complex program transcriptional regulation ctl differentiation',\n",
       " 'collectively data provide evidence runx together box factor orchestrates complex program transcriptional regulation differentiating ctl fig c',\n",
       " 'runx present naive cd cell activation',\n",
       " 'represses runx positive role induction eomes granzyme b perforin ifn gamma',\n",
       " 'runx bind promoter putative regulatory region latter three gene suggesting direct effect gene expression',\n",
       " 'additional experiment needed determine whether eomes runx also direct target gene runx',\n",
       " 'surprisingly runx contributed optimal expression tnf il ifn gamma day fig',\n",
       " 'tnf il requirement runx subsides day fig c possibly compensation runx derepressed runx cell fig',\n",
       " 'runx continues required ifn gamma expression even day perhaps role induction eomes expression fig',\n",
       " 'unexpected finding two box transcription factor bet eomes regulated different kinetics cd cell culture condition nonredundant role subsequent expression key effector protein fig c',\n",
       " 'bet needed early confer activated cd cell competence produce ifn gamma upon restimulation function le important later time',\n",
       " 'eomes induced late function downstream runx may substitute bet promoting acute expression ifn gamma restimulated ctls',\n",
       " 'indeed bet eomes contribute perforin expression nk cell eomes induces granzyme b effectively bet developing th cell thus relative role box transcription factor vary depending cell type',\n",
       " 'surprisingly however eomes bet appeared nonredundant ability induce two marker ctl function prf gzmb fig c',\n",
       " 'rather bet eomes involved regulating granzyme b perforin expression respectively regulation bet eomes mrna protein closely preceded regulation gzmb prf mrna protein respectively',\n",
       " 'bet role perforin expression culture condition eo vp affect granzyme b expression expressed bet runx cell',\n",
       " 'conventional eomes deficient mouse die precursor cell isolated bone marrow transfer cell conditionally deficient eomes recently described unable introduce runx eomes deficient cd cell test formally whether eomes cooperated runx induce perforin expression',\n",
       " 'collectively data consistent transcriptional network preexisting runx cooperates induced box factor bet eomes il rbeta signal unpublished data orchestrate ctl differentiation fig c',\n",
       " 'data recall feed forward interaction bet runx previously described cd th cell distinct two respect differentiating th cell bet induced tcr signal ifn gamma turn induces runx whereas differentiating cd cell preexisting runx required induce box transcription factor eomes',\n",
       " 'whole genome experiment system required establish whether cooperation box runx family transcription factor general feature cellular differentiation program',\n",
       " 'antibody reagent',\n",
       " 'following antibody used intracellular surface stain obtained ebioscience anti il anti ifn gamma anti tnf anti granzyme b anti cd anti cd anti cd',\n",
       " 'anti cd purchased bd',\n",
       " 'chip experiment anti eomes antibody obtained abcam anti runx antibody produced groner laboratory',\n",
       " 'following antibody used immunoblotting anti perforin abcam anti eomes abcam anti pol ii santa cruz biotechnology inc',\n",
       " 'bet antibody provided l',\n",
       " 'glimcher harvard school public health boston',\n",
       " 'following reagent used experiment presented report annexin v fitc apoptosis detection kit bd cd negative isolation kit invitrogen cd microbeads miltenyi biotec sybr green pcr core reagent applied biosystems',\n",
       " 'gp peptide kavynfatc synthesized tuft university core facility mm stock solution prepared dmso',\n",
       " 'isolation culture primary cd cell',\n",
       " 'cd cell wk old tcra x p tcr transgenic taconic c bl j wt tbx jackson laboratory mouse purified purity negative selection invitrogen pooled spleen lymph node cell',\n",
       " 'cd cell runx mouse icr background purified positive selection miltenyi biotec',\n",
       " 'mouse maintained specific pathogen free barrier facility used according protocol approved immune disease institute harvard medical school animal care use committee',\n",
       " 'stimulation purified cd cell cultured cell ml ml flask coated mug ml anti cd clone c anti cd clone pretreatment mug ml goat anti hamster igg',\n",
       " 'h cell removed tcr stimulation recultured concentration x cell ml medium supplemented u ml rhil',\n",
       " 'every h viable cell counted readjusted x cell ml fresh medium containing corresponding amount rhil',\n",
       " 'isolation cd cell runx mouse',\n",
       " 'runx deficient cell fail silence cd expression normally fig',\n",
       " 'therefore fractionated positively selected cd cell runx ko mouse cd cd sp cd cd dp cell separation using anti cd magnetic bead',\n",
       " 'yielded runx ko sp enriched population contained cd cd cell ko dp enriched population contained cd cd cell fig',\n",
       " 'cell stimulated anti cd anti cd removing tcr stimulus culturing medium containing u ml il',\n",
       " 'previously reported tcr induced proliferation runx cd cell severely impaired irrespective cd expression fig',\n",
       " 'however runx cell showed cell surface expression pattern indicative activated cell including regulation cd cd fig',\n",
       " 'expected ability regulate cd runx cd cell responded il supplementation day efficiently expanded day culture period albeit slower rate compared wt cell fig',\n",
       " 'although fraction ko dp cell silenced cd expression activation ratio sp dp cell enriched population remained constant thereafter observe major difference two population throughout culture period indicating term effector ctl differentiation culture condition runx cd cell also coexpress cd indistinguishable',\n",
       " 'data presented fig',\n",
       " 'runx ko sp cell whereas shown fig',\n",
       " 'total runx ko cd cell',\n",
       " 'facs based cytotoxicity assay',\n",
       " 'measure cytotoxicity el thymoma target cell loaded mum gp peptide h h coincubation p cd cell effector target ratio indicated figure well round bottom plate',\n",
       " 'coincubation period cell stained annexin v fitc anti cd allophycocyanin',\n",
       " 'data analysis performed flowjo software tree star inc el target cell cd negative event gated percentage annexin v target cell determined',\n",
       " 'cytokine surface marker staining',\n",
       " 'ass cytokine production cell restimulated nm pma mum ionomycin h unless indicated otherwise figure intracellular cytokine stain performed previously described',\n",
       " 'detect expression surface molecule cell washed pb resuspended facs wash buffer fbs sodium azide mm hepes x pb containing antibody indicated figure previously optimized concentration incubated min room temperature rt washed resuspended formaldehyde fixative solution acquisition facscalibur bd',\n",
       " 'retroviral transduction primary cd cell',\n",
       " 'transduction experiment viral supernatant generated calcium phosphate transfection phoenix cell concentration overnight centrifugation g',\n",
       " 'approximately h initial tcr activation cd cell per well well plate culture medium removed replaced complete medium supplemented mug ml polybrene containing fresh plus concentrated virus',\n",
       " 'plate centrifuged g h rt returning degreesc additional h',\n",
       " 'retroviral construct eomes vp mig control empty vector gift l',\n",
       " 'reiner university pennsylvania philadelphia pa',\n",
       " 'chip real time pcr analysis',\n",
       " 'x cd cell per immunoprecipitation fixed adding th volume fixation solution formaldehyde mm nacl mm edta mm egta mm hepes volume culture medium incubated min rt',\n",
       " 'fixation stopped mm glycine ice min',\n",
       " 'fixed cell washed x cold pb x cold solution mm tris ph mm edta mm egta triton x x cold solution ii mm tris ph mm edta mm egta mm nacl',\n",
       " 'wash cell pellet resuspended x cell ml chip lysis buffer mm nacl mm tris ph triton x sd deoxycholate plus protease phosphatase inhibitor chromatin sheared sonicator yield kb dna fragment',\n",
       " 'preclearing sheared chromatin protein sepharose bead removing input chromatin immunoprecipitation performed adding optimized antibody amount per x cell equivalent mug anti eomes dilution anti runx followed overnight incubation degreesc protein sepharose bead added last h incubation period',\n",
       " 'bead washed x ripa buffer mm tris ph mm nacl mm edta np sd deoxycholate x high salt buffer mm tris ph mm nacl mm edta np sd x te buffer',\n",
       " 'last wash dna eluted resuspending bead elution buffer sd mm nahco',\n",
       " 'input chip chromatin treated rnase mug total h degreesc followed addition proteinase k mug total overnight incubation degreesc reverse cross linking',\n",
       " 'dna purified qiaquick column gel extraction kit qiagen according manufacturer instruction resuspended mul volume',\n",
       " 'real time pcr detection immunoprecipitated target using sybr green pcr kit standard curve obtained serial dilution input dna sample mul chip dna used per pcr reaction performed duplicate',\n",
       " 'melt curve agarose gel analyzed ensure amplification specific target sequence',\n",
       " 'refer table available http www jem org cgi content full jem dc list primer set',\n",
       " 'data presented number immunoprecipitated target sequence relative input chromatin assuming two copy target sequence per cell equivalent used chip',\n",
       " 'northern western blot analysis',\n",
       " 'rna isolation northern blot analysis performed previously described',\n",
       " 'brief mug total rna loaded per lane transferred positively charged nylon membrane hybond n ge healthcare confirmed ethidium bromide staining ribosomal rna specie membrane',\n",
       " 'membrane hybridized ng ml alpha p dctp labeled trichloroacetic acid precipitable probe expresshyb hybridization buffer clontech laboratory inc',\n",
       " 'cdna probe confirmed appropriate single copy specificity condition using genomic southern blot analysis',\n",
       " 'band intensity acquired phosphorimaging analysis',\n",
       " 'western analysis whole cell protein lysates obtained cd cell time point indicated figure clonal expansion u ml il lysis buffer mm tris ph mm nacl glycerol mm edta np resuspending sample mul per cell incubating ice min presence protease inhibitor',\n",
       " 'immunoblot analysis performed antibody indicated figure sd page mug total protein loaded per well',\n",
       " 'quantification detected protein performed intelligent dark box unit la fujifilm normalized loading amount rna pol ii detected lane',\n",
       " 'online supplemental material',\n",
       " 'fig show characterization peripheral cd cell runx mouse',\n",
       " 'fig',\n",
       " 'show effector protein expression runx wt ko cell day vitro culture',\n",
       " 'primer sequence used chip experiment shown table',\n",
       " 'online supplemental material available http www jem org cgi content full jem dc',\n",
       " 'supplementary material',\n",
       " 'supplemental material index',\n",
       " 'kinetics gene expression cd cell differentiation',\n",
       " 'kinetics prf gzmb tbx bet eomes mrna expression differentiating p cd cell analyzed northern blotting',\n",
       " 'rna day th cell used control',\n",
       " 'size mrna transcript indicated',\n",
       " 'b quantification relative mrna amount phosphorimager analysis',\n",
       " 'c kinetics protein expression differentiating p cd cell analyzed immunoblotting',\n",
       " 'size protein band indicated',\n",
       " 'relative protein amount quantified western blot',\n",
       " 'e intracellular staining granzyme b ifn gamma tnf',\n",
       " 'granzyme b staining specific relative isotype control depicted',\n",
       " 'cell restimulated pma ionomycin h',\n",
       " 'f facs based assay measure cytolytic activity p cd cell el target loaded mum gp peptide effector target ratio',\n",
       " 'percentage annexin v apoptotic target cell cd negative el target population dot plot determined histogram',\n",
       " 'cytolytic activity blocked incubation mm egta depicted confirming involvement granule exocytosis perforin granzyme b pathway',\n",
       " 'data representative least five e three f independent experiment',\n",
       " 'regulation perforin granzyme b ifn gamma expression bet eomes differentiating ctls',\n",
       " 'ifn gamma expression wt tbx bet deficient tbx cell',\n",
       " 'naive cd cell cell activated cultured restimulated pma ionomycin h ifn gamma expression assessed intracellular staining',\n",
       " 'number show percentage ifn gamma cell',\n",
       " 'b northern blot analysis prf gzmb mrna expression wt bet deficient cd cell activated either left uninfected uninf transduced retrovirus expressing eomes vp eo vp empty ire gfp cassette gfp',\n",
       " 'total cellular rna analyzed day culture',\n",
       " 'frequency transduced cell culture equivalent construct approximately gfp cell depicted',\n",
       " 'c granzyme b ifn gamma expression tbx tbx cell analyzed restimulated cell cultured',\n",
       " 'ifn gamma production cell transduced eo vp control gfp retrovirus rv measured day h restimulation pma ionomycin',\n",
       " 'number show percentage gfp ifn gamma cell',\n",
       " 'result representative three c two b independent experiment',\n",
       " 'key role runx effector ctl differentiation',\n",
       " 'western analysis runx eomes bet perforin expression runx versus runx cd sp cell differentiated',\n",
       " 'beta actin used loading control',\n",
       " ...]"
      ]
     },
     "execution_count": 89,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "corpus"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 91,
   "id": "b589cc11",
   "metadata": {},
   "outputs": [],
   "source": [
    "from sklearn.feature_extraction.text import TfidfVectorizer\n",
    "Tf = TfidfVectorizer()\n",
    "X = Tf.fit_transform(corpus).toarray()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 110,
   "id": "e5d3c0bf",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "0    Negative_regulation\n",
       "1             Regulation\n",
       "2    Positive_regulation\n",
       "3             Regulation\n",
       "4             Regulation\n",
       "Name: EventType, dtype: object"
      ]
     },
     "execution_count": 110,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "## Handling categorical variable using target guided ordinal encoding\n",
    "y.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 111,
   "id": "5f62ab61",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "Protein_catabolism       20\n",
       "Phosphorylation          52\n",
       "Localization             89\n",
       "Transcription           187\n",
       "Binding                 439\n",
       "Regulation              445\n",
       "Negative_regulation     521\n",
       "Gene_expression         785\n",
       "Positive_regulation    9488\n",
       "Name: EventType, dtype: int64"
      ]
     },
     "execution_count": 111,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "y.value_counts().sort_values()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 112,
   "id": "7141eadb",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "Index(['Protein_catabolism', 'Phosphorylation', 'Localization',\n",
       "       'Transcription', 'Binding', 'Regulation', 'Negative_regulation',\n",
       "       'Gene_expression', 'Positive_regulation'],\n",
       "      dtype='object')"
      ]
     },
     "execution_count": 112,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "y.value_counts().sort_values().index"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 113,
   "id": "86b58a91",
   "metadata": {},
   "outputs": [],
   "source": [
    "ordinal_labels = y.value_counts().sort_values().index"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 98,
   "id": "7c2e1976",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "<enumerate object at 0x0000028D4F76D800>\n"
     ]
    }
   ],
   "source": [
    "enumerate(ordinal_labels,1)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 114,
   "id": "29cb251b",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'Protein_catabolism': 1,\n",
       " 'Phosphorylation': 2,\n",
       " 'Localization': 3,\n",
       " 'Transcription': 4,\n",
       " 'Binding': 5,\n",
       " 'Regulation': 6,\n",
       " 'Negative_regulation': 7,\n",
       " 'Gene_expression': 8,\n",
       " 'Positive_regulation': 9}"
      ]
     },
     "execution_count": 114,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "ordinal_labels2={k:i for i,k in enumerate(ordinal_labels,1)}\n",
    "ordinal_labels2"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 115,
   "id": "24725431",
   "metadata": {},
   "outputs": [],
   "source": [
    "y = y.map(ordinal_labels2)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 116,
   "id": "b906c3c8",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "((12026, 8992), (12026,))"
      ]
     },
     "execution_count": 116,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "X.shape, y.shape"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 117,
   "id": "934dfcbf",
   "metadata": {},
   "outputs": [],
   "source": [
    "## Train Test Split\n",
    "from sklearn.model_selection import train_test_split\n",
    "X_train,X_test,y_train,y_test = train_test_split(X,y,test_size=0.30,random_state=42)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 118,
   "id": "f15d4ba3",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "((8418, 8992), (3608, 8992), (8418,), (3608,))"
      ]
     },
     "execution_count": 118,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "X_train.shape,X_test.shape,y_train.shape,y_test.shape"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 119,
   "id": "32883e3b",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "MultinomialNB()"
      ]
     },
     "execution_count": 119,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "## Naive Baye's Classifier\n",
    "from sklearn.naive_bayes import MultinomialNB\n",
    "naive = MultinomialNB()\n",
    "naive.fit(X_train,y_train)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 120,
   "id": "9a874877",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "0.7827050997782705"
      ]
     },
     "execution_count": 120,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "from sklearn.metrics import accuracy_score\n",
    "y_pred = naive.predict(X_test)\n",
    "acc = accuracy_score(y_pred,y_test)\n",
    "acc"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "70bd3677",
   "metadata": {},
   "source": [
    "## Multinomial Classifier with HyperParameter"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 132,
   "id": "17886fdd",
   "metadata": {},
   "outputs": [],
   "source": [
    "classifier = MultinomialNB(alpha=0.1)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 135,
   "id": "89eb3323",
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "C:\\Users\\ICON\\anaconda3\\lib\\site-packages\\sklearn\\naive_bayes.py:508: UserWarning: alpha too small will result in numeric errors, setting alpha = 1.0e-10\n",
      "  warnings.warn('alpha too small will result in numeric errors, '\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Alpha: 0.0, score : 0.7657982261640798\n",
      "Alpha: 0.1, score : 0.772450110864745\n",
      "Alpha: 0.2, score : 0.7768847006651884\n",
      "Alpha: 0.30000000000000004, score : 0.779379157427938\n",
      "Alpha: 0.4, score : 0.7796563192904656\n",
      "Alpha: 0.5, score : 0.7810421286031042\n",
      "Alpha: 0.6000000000000001, score : 0.7818736141906873\n",
      "Alpha: 0.7000000000000001, score : 0.782150776053215\n",
      "Alpha: 0.8, score : 0.7824279379157428\n",
      "Alpha: 0.9, score : 0.7824279379157428\n"
     ]
    }
   ],
   "source": [
    "import numpy as np\n",
    "previous_score=0\n",
    "for alpha in np.arange(0,1,0.1):\n",
    "    sub_classifier=MultinomialNB(alpha=alpha)\n",
    "    sub_classifier.fit(X_train,y_train)\n",
    "    y_pred = sub_classifier.predict(X_test)\n",
    "    score = accuracy_score(y_pred,y_test)\n",
    "    if score > previous_score:\n",
    "        classifier = sub_classifier\n",
    "    print(\"Alpha: {}, score : {}\".format(alpha,score))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "d96ebc4c",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "markdown",
   "id": "b0a6d40b",
   "metadata": {},
   "source": [
    "## Passive Aggressive Classifier Algorithm"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 121,
   "id": "71ebe0a0",
   "metadata": {},
   "outputs": [],
   "source": [
    "from sklearn.linear_model import PassiveAggressiveClassifier\n",
    "linear_clf = PassiveAggressiveClassifier()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 122,
   "id": "582b4c07",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "0.707039911308204"
      ]
     },
     "execution_count": 122,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "linear_clf.fit(X_train,y_train)\n",
    "y_pred = linear_clf.predict(X_test)\n",
    "acc = accuracy_score(y_pred,y_test)\n",
    "acc"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "ed15a74a",
   "metadata": {},
   "source": [
    "## Random forest Classifier"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 123,
   "id": "2bfdfa55",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "0.7854767184035477"
      ]
     },
     "execution_count": 123,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "from sklearn.ensemble import RandomForestClassifier\n",
    "random = RandomForestClassifier()\n",
    "random.fit(X_train,y_train)\n",
    "y_pred = random.predict(X_test)\n",
    "acc = accuracy_score(y_pred,y_test)\n",
    "acc"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 124,
   "id": "3b8e08de",
   "metadata": {},
   "outputs": [],
   "source": [
    "## Hyperparameter tuning\n",
    "from sklearn.model_selection import GridSearchCV,RandomizedSearchCV"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 130,
   "id": "25e3a773",
   "metadata": {},
   "outputs": [],
   "source": [
    "param_grid = {\n",
    "    'n_estimators':[25,50,100,150],\n",
    "    'max_depth': [3,6,9],\n",
    "    'max_leaf_nodes': [3,6,9]\n",
    "}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 131,
   "id": "9c3e8fb0",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "0.7832594235033259"
      ]
     },
     "execution_count": 131,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "grid_search = GridSearchCV(RandomForestClassifier(),param_grid=param_grid)\n",
    "grid_search.fit(X_train,y_train)\n",
    "y_pred = grid_search.predict(X_test)\n",
    "acc = accuracy_score(y_pred,y_test)\n",
    "acc"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "b926fc5d",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "db3cdd9f",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "c505dd74",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "e5ed71af",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.7"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
